<!DOCTYPE html>
<html lang="en">
    <head>
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <link rel="stylesheet" type="text/css" href="../../css/style.css">

        <title>The National Centre for Text Mining: A Vision for the Future - Ariadne</title>

        <meta name="citation_journal_title" content="Ariadne" />
        <meta name="citation_issn" content="1361-3200">

        
            <meta name="citation_title" content="The National Centre for Text Mining: A Vision for the Future" />
        

        
            <meta name="citation_publication_date" content="2007" />
        

        
            <meta name="citation_issue" content="53" />
        


        
            
                <meta name="citation_author" content="sophia ananiadou" />
            
        

        
            <link rel="alternate" type="application/x-bibtex" href="ananiadou/citation.bibtex.html" title="Ariadne" />
        
            <link rel="alternate" type="application/x-research-info-systems" href="ananiadou/citation.ris" title="Ariadne" />
        

        
    </head>
    <body>
        <header>
            <a id="logo" href="../../index.html"><img src="../../images/logo.png" alt="Home"></a>
            <div id="subtitle">Web Magazine for Information Professionals</div>
            <nav>
                
                    <a href="../../index.html">Home</a>
                
                    <a href="../../current-issue.html">Coming issue</a>
                
                    <a href="../../issue.html">Archive</a>
                
                    <a href="../../author.html">Authors</a>
                
                    <a href="../../articles.html">Articles</a>
                
                    <a href="../../guidelines.html">Guidelines</a>
                
            </nav>
        </header>


<article>
    <header>
        <h1>The National Centre for Text Mining: A Vision for the Future</h1>
    </header>


    
    <aside>
        
            
                <h3>Buzz</h3>
                <p id="buzz">
                    
                        
                        
                            <a href="../../buzz/data/index.html">data</a>
                        
                    
                        
                        
                            <a href="../../buzz/software/index.html">software</a>
                        
                    
                        
                        
                            <a href="../../buzz/framework/index.html">framework</a>
                        
                    
                        
                        
                    
                        
                        
                    
                        
                        
                            <a href="../../buzz/database/index.html">database</a>
                        
                    
                        
                        
                    
                        
                        
                    
                        
                        
                            <a href="../../buzz/portal/index.html">portal</a>
                        
                    
                        
                        
                    
                        
                        
                            <a href="../../buzz/usability/index.html">usability</a>
                        
                    
                        
                        
                            <a href="../../buzz/infrastructure/index.html">infrastructure</a>
                        
                    
                        
                        
                    
                        
                        
                            <a href="../../buzz/metadata/index.html">metadata</a>
                        
                    
                        
                        
                            <a href="../../buzz/doi/index.html">doi</a>
                        
                    
                        
                        
                            <a href="../../buzz/identifier/index.html">identifier</a>
                        
                    
                        
                        
                            <a href="../../buzz/vocabularies/index.html">vocabularies</a>
                        
                    
                        
                        
                            <a href="../../buzz/blog/index.html">blog</a>
                        
                    
                        
                        
                            <a href="../../buzz/repositories/index.html">repositories</a>
                        
                    
                        
                        
                    
                        
                        
                    
                        
                        
                    
                        
                        
                            <a href="../../buzz/visualisation/index.html">visualisation</a>
                        
                    
                        
                        
                    
                        
                        
                            <a href="../../buzz/e-learning/index.html">e-learning</a>
                        
                    
                        
                        
                    
                        
                        
                            <a href="../../buzz/ontologies/index.html">ontologies</a>
                        
                    
                        
                        
                    
                        
                        
                    
                        
                        
                    
                        
                        
                            <a href="../../buzz/e-science/index.html">e-science</a>
                        
                    
                        
                        
                            <a href="../../buzz/e-research/index.html">e-research</a>
                        
                    
                        
                        
                    
                        
                        
                            <a href="../../buzz/uima/index.html">uima</a>
                        
                    
                        
                        
                    
                        
                        
                            <a href="../../buzz/interoperability/index.html">interoperability</a>
                        
                    
                        
                        
                    
                        
                        
                            <a href="../../buzz/research/index.html">research</a>
                        
                    
                        
                        
                    
                </p>
            
        

        <h3>Citation</h3>
        <ul>
            <li><a href="ananiadou/citation.bibtex.html">BibTex</a></li>
            <li><a href="ananiadou/citation.ris">RIS</a></li>
        </ul>
    </aside>
    <div class="summary">
        <a href="http://www.ariadne.ac.uk/issue53/ananiadou#author1">Sophia Ananiadou</a> describes NaCTeM and the main scientific challenges it helps to solve together with issues related to deployment, use and uptake of NaCTeM's text mining tools and services.
    </div>

    <div class="content">
        <p>One of the defining challenges of e-Science is dealing with the data deluge [<a href="ananiadou.html#1">1</a>] information overload and information overlook. More than 8,000 scientific papers are published every week (on Google Scholar, for example). Without sophisticated new tools, researchers will be unable to keep abreast of developments in their field and valuable new sources of research data will be under-exploited. The capability of text mining (TM) to find knowledge hidden in text and to present it in a concise form makes it an essential part of any strategy for addressing these problems. As text mining matures, it will increasingly enable researchers to collect, maintain, interpret, curate, and discover knowledge needed for research and education, efficiently and systematically. The National Centre for Text Mining (NaCTeM) is playing a critical role in ensuring that UK researchers are aware of and have access to effective TM solutions, and are able to exploit their capabilities to the full.</p><h2>Text Mining Services for the UK Academic Community</h2><p>There are many kinds of text mining services, with many sub-processes involved, and many types of user in varying domains with different needs, interests, purposes, requirements and degrees of technological aptitude.</p><p>NaCTeM has developed a number of high-quality text mining service exemplars for the UK academic community. The exemplars produced are:</p><p><strong>TerMine, AcroMine</strong> [<a href="ananiadou.html#2">2</a>][<a href="ananiadou.html#3">3</a>] extract candidate terms from text, and map them between biology acronyms and their full forms. TerMine is a foundational service in that terminology is central to many activities and is often a hurdle for humans and language processors alike. Termine is a service for automatic term recognition which identifies the most important terms in a document ranking them according to their significance and proposing potential expansions of all acronyms found. It is based on the C-value method for automatic term recognition [<a href="ananiadou.html#4">4</a>].</p><div align="center"><img alt="screenshot (249KB) : Figure 1: TerMine: A service for automatic term recognition" border="0" height="471" src="../../images/issue53-ananiadou/termine-new.jpg" title="screenshot (249KB) : Figure 1: TerMine: A service for automatic term recognition" width="650"><p class="caption">Figure 1: TerMine: A service for automatic term recognition</p></div><p>AcroMine is based on a novel approach for recognising acronym definitions in a text collection [<a href="ananiadou.html#5">5</a>]. Applied to the whole MEDLINE, it deals with terminological variation, an integral part of the linguistic ability to realise a concept in different ways. This is also an obstacle to information retrieval.</p><div align="center"><img alt="screenshot (120KB) : Figure 2: AcroMine finds expanded forms of acronyms and disambiguates them" border="0" height="400" src="../../images/issue53-ananiadou/acromine-new.jpg" title="screenshot (120KB) : Figure 2: AcroMine finds expanded forms of acronyms and disambiguates them" width="650"><p align="center" class="caption">Figure 2: AcroMine finds expanded forms of acronyms and disambiguates them</p><p class="caption" style="text-align: left;">AcroMine finds expanded forms of acronyms from a database created from the whole of Medline and disambiguates them. AcroMine has also been incorporated into the TerMine user interface. TerMine, moreover, crucially supports Semantic Web activities and has been used as an aid to ontology construction and controlled vocabularies [<a href="ananiadou.html#6">6</a>].</p></div><div align="center"><img alt="screenshot (21KB) : Figure 3: Overall architecture of TerMine" border="0" height="422" src="../../images/issue53-ananiadou/figure03.png" title="screenshot (21KB) : Figure 3: Overall architecture of TerMine" width="686"><p class="caption">Figure 3: Overall architecture of TerMine</p></div><p><strong>Cheshire/TerMine</strong> [<a href="ananiadou.html#7">7</a>] integrates information retrieval (provided by Cheshire) [<a href="ananiadou.html#8">8</a>] and TerMine to offer users a search facility based on terms with which they are familiar, and which moreover retrieves documents according to the importance of the terminology they contain. For example, this service allows the user to search for documents related to a subject area, to find associated terms related with the query and to select the most appropriate documents to view. This service identifies important terms (that the user may not have known) combining the best of search and browse models of information access.</p><p><strong>MEDIE</strong> [<a href="ananiadou.html#9">9</a>] provides real-time semantic information retrieval based on the retrieval of relational concepts from huge texts. It is an intelligent search engine which uses semantic retrieval technologies to identify sentences containing biomedical correlations for queried terms from Medline abstracts. The service runs on the whole of Medline and is based on semantically annotated texts using deep parsing and named entity recognition. Sentences are annotated in advance with semantic structures and stored in a structured database. User requests are converted on the fly into patterns of these semantic annotations, and texts are retrieved by matching these patterns with the pre-computed semantic annotations [<a href="ananiadou.html#10">10</a>].</p><div align="center"><img alt="screenshot (157KB) : Figure 4: Medie: An intelligent search engine using semantic retrieval technologies" border="0" height="284" src="../../images/issue53-ananiadou/medie-new.jpg" title="screenshot (157KB) : Figure 4: Medie: An intelligent search engine using semantic retrieval technologies" width="647"><p class="caption">Figure 4: Medie: An intelligent search engine using semantic retrieval technologies</p></div><p><strong>InfoPubMed </strong>[<a href="ananiadou.html#11">11</a>] extracts and visualises protein-protein interactions. Info-PubMed is an efficient PubMed search tool, helping users to find information about biomedical entities such as genes, proteins, and the interactions between them.</p><div align="center"><img alt="screenshot (161KB) : Figure 5: InfoPubMed provides information and visualisation of biomedical interactions extracted from Medline" border="0" height="307" src="../../images/issue53-ananiadou/infopubmed-new.jpg" title="screenshot (161KB) : Figure 5: InfoPubMed provides information and visualisation of biomedical interactions extracted from Medline" width="649"><p class="caption">Figure 5: InfoPubMed provides information and visualisation of biomedical interactions extracted from Medline</p></div><p>It provides information and visualisation of biomedical interactions extracted from Medline using deep semantic parsing technology. This is supplemented with the GENA term dictionary which consists of over 200,000 protein/gene names and identifications of disease types and organisms</p><p>NaCTeM services are underpinned by a number of generic natural language processing tools which have already been adapted for the domain of biology. Most of these tools are currently being adapted for use in the social sciences within the framework of the ASSERT [<a href="ananiadou.html#12">12</a>][<a href="ananiadou.html#13">13</a> Project whose aim is to provide automated assistance to social scientists undertaking systematic reviews of the research literature. Figure 6 shows how text mining techniques are being used to support the stages of searching, screening and synthesising in systematic reviews.</p><div align="center"><img alt="screenshot (16KB) : Figure 6: Text mining technologies for enhancing systematic reviewing" border="0" height="292" src="../../images/issue53-ananiadou/figure-6.gif" title="screenshot (16KB) : Figure 6: Text mining technologies for enhancing systematic reviewing" width="522"><p class="caption">Figure 6: Text mining technologies for enhancing systematic reviewing</p></div><p>Text mining techniques have the potential to revolutionise the way we approach research synthesis but our longer term interest is to understand how we can apply these techniques more widely in a variety of use cases in the social sciences. To achieve this, we will use systematic reviewing to demonstrate the potential of text mining for the social science research community and to establish requirements for a generic toolkit of text mining services which can be integrated into different research practices. This provides its own set of issues for development in terms of interoperability with techniques or software currently employed in the systematic review activity but also with other text mining tools and services used by the social science community. For example a researcher investigating the role of new media in politics could be interested in combining the toolset with Internet news feed or blog readers, their own evidence tracking systems or even other tools for carrying out opinion analysis. We need to ensure that our tools are therefore flexible and robust enough to allow for this, whilst providing sufficient functionality to ensure interoperability between the many formats and standards that this would entail.</p><h2>The NaCTeM Roadmap</h2><p>Based on the current service exemplars described previously, NaCTeM's roadmap in figure 7 below outlines our vision for the next 5 years. This vision situates NaCTeM with respect to:</p><ol><li>the general goals text mining aims to address</li><li>the main scientific challenges it helps to solve, thus providing breakthrough to scientific discoveries across domains, and</li><li>the main issues related to the deployment, use and uptake of NaCTeM text mining solutions and services in respect of a wide variety of users. The roadmap sets out the core technology underpinning the provision of NaCTeM's text mining services and solutions over this period. As our services are user-driven these examples will be further refined and developed in close consultation with our user communities.</li></ol><p><a name="roadmap"></a></p><div align="center"><a href="../../images/issue53-ananiadou/roadmap-large.jpg"><img alt="screenshot (73KB) : Figure 7: The NaCTeM Roadmap (with link in caption to larger version)" border="0" height="440" src="../../images/issue53-ananiadou/figure-7.jpg" title="screenshot (73KB) : Figure 7: The NaCTeM Roadmap (with link in caption to larger version)" width="600"></a><p class="caption">Figure 7: The NaCTeM Roadmap (<a href="../../images/issue53-ananiadou/roadmap-large.jpg" title="Link to larger version of illustration: Roadmap of Text Mining">larger version</a>)</p></div><h3>Issues</h3><p>Full paper processing is necessary for the discovery of new knowledge and evidence from literature, but currently there is limited availability of open access collections for the scientific community and there is uncertainty over IPR (Intellectual Property Rights) in data derived via text mining [<a href="ananiadou.html#14">14</a>], especially as publishers move to new models [<a href="ananiadou.html#15">15</a>][<a href="ananiadou.html#16">16</a>]. Standards are needed for the types of annotation we use to represent layers of linguistic and semantic analysis, and thus there is a need for cooperation among stakeholders. TM tools are increasingly released as open source, but in order to be interoperable, common infrastructure and annotation schemes [<a href="ananiadou.html#17">17</a>] have to be adopted. Large-scale resources (annotated corpora, lexicons, ontologies) [<a href="ananiadou.html#18">18</a>] are required to support TM, but are expensive for one centre to produce (and maintain), requiring collaboration, sharing and co-funding. TM as a new technology faces potential barriers which must be tackled. TM tools must match users' requirements and be usable if the technology is to achieve wide adoption. NaCTeM is working closely with NCeSS [<a href="ananiadou.html#19">19</a>] to ensure that it is in a position to identify barriers to adoption and develop strategies to address them.</p><h3>Text Mining Technology Supporting Service Provision</h3><p>An important aspect of the roadmap is the core technology and computational infrastructure needed to allow NaCTeM text-mining services to be developed for use by the community. Our roadmap lays out the main steps we expect to take. These steps are, in several cases, related to general rather than specific technological development. For example, more generally applicable text mining technology to support large-scale services depends on other developments, e.g. the ability to process full texts [<a href="ananiadou.html#20">20</a>]. In order to process full texts on a large scale to support user services, a pre-requisite is parallelisation. Early steps in being able to deploy our text mining services as Web Services lead to processes involving embedding text mining Web Services in workflows, composition of TM services via Web Services, and so on.</p><p>Development of highly efficient machine-learning techniques for text processing is a pre-requisite for classification of extracted information that forms the core of large-scale (repository-wide) metadata creation. The ability more generally to add semantics to textual data on a large scale supports full-text processing. In order to be able to adapt rapidly to new domains in coming years, adaptive learning technology must be in place. Integration of data mining and text mining on a large scale becomes possible at a later stage as our enabling tools and services are put in place.</p><p>None of the above stages can happen easily without early effort to provide workable levels of interoperability, re-usability and portability. While NaCTeM will continue to offer a portal enabling users to find out about best-of-breed text mining tools, integration of arbitrary tools demands not only a common infrastructure but also a common linguistic annotation scheme [<a href="ananiadou.html#21">21</a>]. Although there are solutions for the former there is no solution for the latter. NaCTeM has actively promoted the idea of interoperability of text mining tools in the wider community. We are already using IBM's UIMA (Unstructured Information Management Architecture) [<a href="ananiadou.html#22">22</a>] to support interoperability. UIMA has a set of useful functionalities, such as type definitions shared by modules, management of complex objects, links between multiple annotations and the original text, and a GUI for module integration. However, it simply provides a framework for, and not the detail of, common annotation schemes. Thus, the user community has to develop its own platforms with a set of actual software modules. In addition, simply wrapping existing modules in UIMA does not offer a complete solution for flexible tool integration, necessary for practical applications in a variety of domains. Users, and this includes both developers and end-users of TM systems, tend to be confused when faced with choosing appropriate modules for their own tasks from a collection of a large number of tools.</p><p>NaCTeM has ensured that its own tools and services are interoperable thus providing users with a coherent interoperable text mining suite. In addition, we have developed a general combinatorial text mining comparator which: generates possible combinations of tools for a specific workflow; compares/evaluates the results; facilitates module integration; and guides the selection of text mining modules [<a href="ananiadou.html#23">23</a>] Processing full texts on a large scale necessitates the development of an appropriate massively parallel data management infrastructure to support the use of the machine learning text mining tools we use to improve the accuracy of our services and the ease of adaptation of our toolkit for new domains. Serious use of text mining services implies the ability and the capacity of such services to handle very large amounts of data if e-scientists are to make the kinds of breakthrough they are hoping for. It is part of the remit of NaCTeM to support such large scale processing. Taken together with the issue of interoperability this further motivates us to provide services around the tools we develop. The integration of TM into a workflow environment (e.g. Taverna, [<a href="ananiadou.html#24">24</a>]) will allow us to reach a wide community of e-scientists and will provide additional functionality to the widely popular workflow environment by augmenting data with evidence and results from the literature.</p><h3>Scientific Challenges NaCTem Services Help to Address</h3><p>The challenge of providing evidence for pathways from literature is currently being tackled within the REFINE Project [<a href="ananiadou.html#25">25</a>]. Our aim is to annotate texts semantically in order to extract biological relations which provide the evidence for the detection and classification of the species and reactions in bio-chemical models.</p><p>Although we cannot provide the full answer to the challenge of extracting quantitative data from qualitative materials which is currently exercising the social sciences, we are currently offering ways of eliminating some of its routine and mechanical aspects [<a href="ananiadou.html#12">12</a>]. Success in this area is a pre-requisite for large-scale harvesting of opinions from qualitative survey data and sophisticated media analysis. The successful use of TM to integrate heterogeneous knowledge sources in biology is leading to the discovery of gene-disease associations [<a href="ananiadou.html#26">26</a>]; applied more widely it will lead to TM ushering in and supporting new, data-driven research modes for e-Science.</p><h3>e-Research and e-Science Goals</h3><p>TM enables the e-researcher and e-scientist in knowledge discovery by finding implicit associations hidden in text. It facilitates search by providing intelligent ways to retrieve information. Semantic data derived from TM support new models of e-publishing. Scientific publications are evolving to allow more collaborative ways of communication via social networks. Rich metadata and citation analysis enriched with TM allow us to do personalised searching, extracting facts of relevance to the user.</p><h2>Conclusion</h2><p>NaCTeM's text mining tools and services offer numerous benefits to a wide range of users. These range from considerable reductions in time and effort for finding and linking pertinent information from large scale textual resources, to customised solutions in semantic data analysis and knowledge management. Enhancing metadata is one of the important benefits of deploying text mining services. TM is being used for subject classification, creation of taxonomies, controlled vocabularies, ontology building and Semantic Web activities. As NaCTeM enters into its second phase we are aiming for improved levels of collaboration with Semantic Grid and Digital Library initiatives and contributions to bridging the gap between the library world and the e-Science world through an improved facility for constructing metadata descriptions from textual descriptions via TM.</p><h2>References</h2><ol><li><a name="1"></a>Hey, AJC &amp; Trefethen, A., "The data deluge: an e-science perspective". In Berman F., Fox GC., Hey AJG (eds) Grid Computing: making the global infrastructure a reality. 2003, Chichester, UK: John Wiley, pp.809-824</li><li><a name="2"></a>Termine Web Demonstrator <a href="http://www.nactem.ac.uk/software/termine/">http://www.nactem.ac.uk/software/termine/</a></li><li><a name="3"></a>Acromine Web Demonstrator <a href="http://www.nactem.ac.uk/software/acromine/">http://www.nactem.ac.uk/software/acromine/</a></li><li><a name="4"></a>Frantzi, K., Ananiadou, S., and Mima, H., "Automatic Recognition of Multi-Word Terms: the C/NC value method". <em>International Journal of Digital Libraries</em>, 2000, vol. 3:2, pp. 115-130.</li><li><a name="5"></a>Okazaki, N. and Ananiadou, S., "Building an Abbreviation Dictionary using a Term Recognition Approach", Bioinformatics 2006 22(24):3089-3095.</li><li><a name="6"></a>Spasi?, I., et al. "Facilitating the development of controlled vocabularies for metabolomics with text mining", 2007, in ISMB/ECCB, Bio-Ontologies SIG Workshop, Vienna, Austria, pp. 103-106</li><li><a name="7"></a>Cheshire3-Termine Demonstration using Medline Abstracts <a href="http://www.nactem.ac.uk/software/ctermine/">http://www.nactem.ac.uk/software/ctermine/</a></li><li><a name="8"></a>Cheshire3 Information Framework <a href="http://www.cheshire3.org/">http://www.cheshire3.org/</a></li><li><a name="9"></a>MEDIE <a href="http://www-tsujii.is.s.u-tokyo.ac.jp/medie/">http://www-tsujii.is.s.u-tokyo.ac.jp/medie/</a></li><li><a name="10"></a>Miyao, Y., et al. "Semantic Retrieval for the Accurate Identification of Relational Concepts in Massive Textbases", 2006, In Proc. COLING-ACL, pp.1017-1024.</li><li><a name="11"></a>Info-PubMed <a href="https://www-tsujii.is.s.u-tokyo.ac.jp/info-pubmed/">https://www-tsujii.is.s.u-tokyo.ac.jp/info-pubmed/</a></li><li><a name="12"></a>ASSERT <a href="http://www.nactem.ac.uk/assert/">http://www.nactem.ac.uk/assert/</a></li><li><a name="13"></a>Ananiadou, S., Procter, R., Rea, B., Sasaki, Y., and Thomas, J., "Supporting Systematic Reviews using Text Mining", 2007, 3rd International Conference on e-Social Science, Ann Arbor.</li><li><a name="14"></a>Korn, N., Oppenheim, C. and Duncan, C., "IPR and Licensing issues in Derived Data", Report submitted to the JISC, April 2007</li><li><a name="15"></a>Doyle H, Gass A, Kennison, R. "Open Access and Scientific Societies" <em>PLoS Biology</em>, 2004, Vol. 2, No. 5, e156: doi:10.1371/journal.pbio.002015<br><a href="http://biology.plosjournals.org/perlserv/?request=get-document&amp;doi=10.1371/journal.pbio.0020156">http://biology.plosjournals.org/perlserv/?request=get-document&amp;doi=10.1371/journal.pbio.0020156</a></li><li><a name="16"></a>Rowlands, I, Nicholas D., "New journal publishing models: An international survey of senior researchers". 2005 <a href="http://www.slais.ucl.ac.uk/papers/dni-20050925.pdf">http://www.slais.ucl.ac.uk/papers/dni-20050925.pdf</a></li><li><a name="17"></a>Piao, S., Ananiadou S. and McNaught, J., Integrating Annotation Tools into UIMA for Interoperability, 2007, Sixth UK e-Science All Hands Meeting (AHM2007)</li><li><a name="18"></a>J. D. Kim, T. Ohta, Y. Tateishi, and J. Tsujii, "GENIA corpus - a semantically annotated corpus for biotext mining", Bioinformatics,19(suppl. 1):i180-i182 2003.</li><li><a name="19"></a>e-Research Community: About the Enabling Uptake of e-Infrastructure Services Project <a href="http://www.e-researchcommunity.org/projects/e-uptake/">http://www.e-researchcommunity.org/projects/e-uptake/</a></li><li><a name="20"></a>Tekiner, F. and Ananiadou, S. Towards Text Mining Terabytes of Text Documents, 2007 Microsoft e-Science Workshop <a href="https://www.mses07.net/Main.aspx">https://www.mses07.net/Main.aspx</a></li><li><a name="21"></a>Hahn, U., Buyko, E., Tomanek, K., Piao, S., McNaught, J., Tsuruoka, Y. and Ananiadou, S. "An Annotation Type System for a Data-Driven NLP Pipeline." 2007, The Linguistic Annotation Workshop (LAW) of ACL 2007 <a href="http://www.ling.uni-potsdam.de/acl-lab/LAW-07.html">http://www.ling.uni-potsdam.de/acl-lab/LAW-07.html</a></li><li><a name="22"></a>Lally, A., and Ferrucci, D., "Building an Example Application with the Unstructured Information Management Architecture", 2004, IBM Systems Journal 43, No. 3, 455-475.</li><li><a name="23"></a>Kano, Y., Nguyen, N., Saetre, R., Yoshida, K., Miyao, Y., Tsuruoka, Y., Matsubayashi, Y., Ananiadou, S. and Tsujii, J. (2008) Filling the gaps between tools and users: a tool comparator, using protein-protein interaction as an example, in PSB 2008, Hawaii <a href="http://psb.stanford.edu/">http://psb.stanford.edu/</a></li><li><a name="24"></a>Taverna Project Web site <a href="http://taverna.sourceforge.net/">http://taverna.sourceforge.net/</a></li><li><a name="25"></a>REFINE Project: Representing Evidence For Interacting Network Elements <a href="http://dbkgroup.org/refine/">http://dbkgroup.org/refine/</a></li><li><a name="26"></a>Chun, H., Tsuruoka, Y., Kim, J.D, Shiba, R., Nagata, N., Hishiki, T. and Tsujii, J. "Automatic Recognition of Topic-Classified Relations between Prostate Cancer and Genes using MEDLINE Abstracts", 2006, BMC-Bioinformatics. 7 (Suppl 3).</li></ol><h2>Author Details</h2><p class="authors"><a name="author1"></a> <strong>Sophia Ananiadou</strong><br>Reader in Text Mining<br>University of Manchester<br>School of Computer Science<br>National Centre for Text Mining<br>131 Princess St<br>Manchester<br>M1 7DN<br>UK<br><br>Email: <a href="mailto:Sophia.Ananiadou@manchester.ac.uk">Sophia.Ananiadou@manchester.ac.uk</a><br>Web site: <a href="http://www.nactem.ac.uk/">http://www.nactem.ac.uk/</a><br>Web site: <a href="http://personalpages.manchester.ac.uk/staff/sophia.ananiadou/">http://personalpages.manchester.ac.uk/staff/sophia.ananiadou/</a></p><p class="right"><a href="ananiadou.html#top"><small>Return to top</small></a></p><p class="footer">Article Title: "The National Centre for Text Mining: A Vision for the Future"<br>Author: Sophia Ananiadou<br>Publication Date: 30-October-2007 Publication: Ariadne Issue 53<br>Originating URL: http://www.ariadne.ac.uk/issue53/ananiadou/</p>

    </div>

    <footer>
        
<a class="resp-sharing-button__link" href="https://facebook.com/sharer/sharer.php?u=http%3A%2F%2Fsharingbuttons.io" target="_blank" aria-label="Facebook">
    <div class="resp-sharing-button resp-sharing-button--facebook resp-sharing-button--medium"><div aria-hidden="true" class="resp-sharing-button__icon resp-sharing-button__icon--solidcircle">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 0C5.38 0 0 5.38 0 12s5.38 12 12 12 12-5.38 12-12S18.62 0 12 0zm3.6 11.5h-2.1v7h-3v-7h-2v-2h2V8.34c0-1.1.35-2.82 2.65-2.82h2.35v2.3h-1.4c-.25 0-.6.13-.6.66V9.5h2.34l-.24 2z"/></svg></div>Facebook</div>
</a>


<a class="resp-sharing-button__link" href="https://twitter.com/intent/tweet/?text=Super%20fast%20and%20easy%20Social%20Media%20Sharing%20Buttons.%20No%20JavaScript.%20No%20tracking.&amp;url=http%3A%2F%2Fsharingbuttons.io" target="_blank" aria-label="Twitter">
    <div class="resp-sharing-button resp-sharing-button--twitter resp-sharing-button--medium"><div aria-hidden="true" class="resp-sharing-button__icon resp-sharing-button__icon--solidcircle">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 0C5.38 0 0 5.38 0 12s5.38 12 12 12 12-5.38 12-12S18.62 0 12 0zm5.26 9.38v.34c0 3.48-2.64 7.5-7.48 7.5-1.48 0-2.87-.44-4.03-1.2 1.37.17 2.77-.2 3.9-1.08-1.16-.02-2.13-.78-2.46-1.83.38.1.8.07 1.17-.03-1.2-.24-2.1-1.3-2.1-2.58v-.05c.35.2.75.32 1.18.33-.7-.47-1.17-1.28-1.17-2.2 0-.47.13-.92.36-1.3C7.94 8.85 9.88 9.9 12.06 10c-.04-.2-.06-.4-.06-.6 0-1.46 1.18-2.63 2.63-2.63.76 0 1.44.3 1.92.82.6-.12 1.95-.27 1.95-.27-.35.53-.72 1.66-1.24 2.04z"/></svg></div>Twitter</div>
</a>


<a class="resp-sharing-button__link" href="https://plus.google.com/share?url=http%3A%2F%2Fsharingbuttons.io" target="_blank" aria-label="Google+">
    <div class="resp-sharing-button resp-sharing-button--google resp-sharing-button--medium"><div aria-hidden="true" class="resp-sharing-button__icon resp-sharing-button__icon--solidcircle">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12.65 8.6c-.02-.66-.3-1.3-.8-1.8S10.67 6 9.98 6c-.63 0-1.2.25-1.64.68-.45.44-.68 1.05-.66 1.7.02.68.3 1.32.8 1.8.96.97 2.6 1.04 3.5.14.45-.45.7-1.05.67-1.7zm-2.5 5.63c-2.14 0-3.96.95-3.96 2.1 0 1.12 1.8 2.08 3.94 2.08s3.98-.93 3.98-2.06c0-1.14-1.82-2.1-3.98-2.1z"/><path d="M12 0C5.38 0 0 5.38 0 12s5.38 12 12 12 12-5.38 12-12S18.62 0 12 0zm-1.84 19.4c-2.8 0-4.97-1.35-4.97-3.08s2.15-3.1 4.94-3.1c.84 0 1.6.14 2.28.36-.57-.4-1.25-.86-1.3-1.7-.26.06-.52.1-.8.1-.95 0-1.87-.38-2.57-1.08-.67-.68-1.06-1.55-1.1-2.48-.02-.94.32-1.8.96-2.45.65-.63 1.5-.93 2.4-.92V5h3.95v1h-1.53l.12.1c.67.67 1.06 1.55 1.1 2.48.02.93-.32 1.8-.97 2.45-.16.15-.33.3-.5.4-.2.6.05.8.83 1.33.9.6 2.1 1.42 2.1 3.56 0 1.73-2.17 3.1-4.96 3.1zM20 10h-2v2h-1v-2h-2V9h2V7h1v2h2v1z"/></svg></div>Google+</div>
</a>


<a class="resp-sharing-button__link" href="mailto:?subject=Super%20fast%20and%20easy%20Social%20Media%20Sharing%20Buttons.%20No%20JavaScript.%20No%20tracking.&amp;body=http%3A%2F%2Fsharingbuttons.io" target="_self" aria-label="E-Mail">
    <div class="resp-sharing-button resp-sharing-button--email resp-sharing-button--medium"><div aria-hidden="true" class="resp-sharing-button__icon resp-sharing-button__icon--solidcircle">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 0C5.38 0 0 5.38 0 12s5.38 12 12 12 12-5.38 12-12S18.62 0 12 0zm8 16c0 1.1-.9 2-2 2H6c-1.1 0-2-.9-2-2V8c0-1.1.9-2 2-2h12c1.1 0 2 .9 2 2v8z"/><path d="M17.9 8.18c-.2-.2-.5-.24-.72-.07L12 12.38 6.82 8.1c-.22-.16-.53-.13-.7.08s-.15.53.06.7l3.62 2.97-3.57 2.23c-.23.14-.3.45-.15.7.1.14.25.22.42.22.1 0 .18-.02.27-.08l3.85-2.4 1.06.87c.1.04.2.1.32.1s.23-.06.32-.1l1.06-.9 3.86 2.4c.08.06.17.1.26.1.17 0 .33-.1.42-.25.15-.24.08-.55-.15-.7l-3.57-2.22 3.62-2.96c.2-.2.24-.5.07-.72z"/></svg></div>E-Mail</div>
</a>


<a class="resp-sharing-button__link" href="https://pinterest.com/pin/create/button/?url=http%3A%2F%2Fsharingbuttons.io&amp;media=http%3A%2F%2Fsharingbuttons.io&amp;description=Super%20fast%20and%20easy%20Social%20Media%20Sharing%20Buttons.%20No%20JavaScript.%20No%20tracking." target="_blank" aria-label="Pinterest">
    <div class="resp-sharing-button resp-sharing-button--pinterest resp-sharing-button--medium"><div aria-hidden="true" class="resp-sharing-button__icon resp-sharing-button__icon--solidcircle">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 0C5.38 0 0 5.38 0 12s5.38 12 12 12 12-5.38 12-12S18.62 0 12 0zm1.4 15.56c-1 0-1.94-.53-2.25-1.14l-.65 2.52c-.4 1.45-1.57 2.9-1.66 3-.06.1-.2.07-.22-.04-.02-.2-.32-2 .03-3.5l1.18-5s-.3-.6-.3-1.46c0-1.36.8-2.37 1.78-2.37.85 0 1.25.62 1.25 1.37 0 .85-.53 2.1-.8 3.27-.24.98.48 1.78 1.44 1.78 1.73 0 2.9-2.24 2.9-4.9 0-2-1.35-3.5-3.82-3.5-2.8 0-4.53 2.07-4.53 4.4 0 .5.1.9.25 1.23l-1.5.82c-.36-.64-.54-1.43-.54-2.28 0-2.6 2.2-5.74 6.57-5.74 3.5 0 5.82 2.54 5.82 5.27 0 3.6-2 6.3-4.96 6.3z"/></svg></div>Pinterest</div>
</a>


<a class="resp-sharing-button__link" href="https://www.linkedin.com/shareArticle?mini=true&amp;url=http%3A%2F%2Fsharingbuttons.io&amp;title=Super%20fast%20and%20easy%20Social%20Media%20Sharing%20Buttons.%20No%20JavaScript.%20No%20tracking.&amp;summary=Super%20fast%20and%20easy%20Social%20Media%20Sharing%20Buttons.%20No%20JavaScript.%20No%20tracking.&amp;source=http%3A%2F%2Fsharingbuttons.io" target="_blank" aria-label="LinkedIn">
    <div class="resp-sharing-button resp-sharing-button--linkedin resp-sharing-button--medium"><div aria-hidden="true" class="resp-sharing-button__icon resp-sharing-button__icon--solidcircle">
    <svg version="1.1" x="0px" y="0px" width="24px" height="24px" viewBox="0 0 24 24" enable-background="new 0 0 24 24" xml:space="preserve">
    <path d="M12,0C5.383,0,0,5.383,0,12s5.383,12,12,12s12-5.383,12-12S18.617,0,12,0z M9.5,16.5h-2v-7h2V16.5z M8.5,7.5 c-0.553,0-1-0.448-1-1c0-0.552,0.447-1,1-1s1,0.448,1,1C9.5,7.052,9.053,7.5,8.5,7.5z M18.5,16.5h-3V13c0-0.277-0.225-0.5-0.5-0.5 c-0.276,0-0.5,0.223-0.5,0.5v3.5h-3c0,0,0.031-6.478,0-7h3v0.835c0,0,0.457-0.753,1.707-0.753c1.55,0,2.293,1.12,2.293,3.296V16.5z" />
    </svg></div>LinkedIn</div>
</a>


    </footer>
</article>

        <footer>
            <p>Ariadne is published by <a href="http://www.lboro.ac.uk/library/" target="_blank">Loughborough University Library</a></p>
            <p>&copy; Ariadne ISSN: 1361-3200. See our explanations of <a href="../../about/copyright.html" title="Explanation of Access Terms and Copyright">Access Terms and Copyright</a> and <a href="../../privacy-statement.html">Privacy Statement</a>.</p>
        </footer>
    </body>
</html>

