<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN"                         "http://www.w3.org/TR/REC-html40/loose.dtd">

<html>

   <!-- Formatting 4/23/04, bw --> 

  <head>     

<title>Determining Space from Place for Natural History Collections: In a Distributed Digital Library Environment</title> 

           <link rel="metadata" href="05beaman.meta.xml"> 

  

 <link rel="stylesheet" type="text/css" href="../style/main.css" title="Default Style Sheet"> 

<style type="text/css">


</style>

                    

<meta name="DOI" content="10.1045/may2004-beaman">            

<meta HTTP-EQUIV="content-type" content="text/html; CHARSET=iso-8859-1">

 

         <meta name="description" content="D-Lib Magazine"> 

          

  <meta name="keywords" content="D-Lib Magazine, Digital Libraries, Digital Library Research">

 </head>   

<body bgcolor="#ffffff"> 

 <div class="center">  

<table width="700" border="0" cellspacing="0" cellpadding="0" align="center">                                          

<tr>                   
<td class="center">      
<p><font style="font-weight: bold;">
<a class="menu" href="../../../Architext/AT-dlib2query.html" target="_top">Search</a> &nbsp; <a name="Top">|</a>  &nbsp;
<a class="menu" href="../../../back.html" target="_top">Back Issues</a> &nbsp; |  &nbsp;            
<a class="menu" href="../../../author-index.html" target="_top">Author Index</a> &nbsp; | &nbsp;          
<a class="menu" href="../../../title-index.html" target="_top">Title Index</a> &nbsp; | &nbsp;           
<a class="menu" href="../05contents.html" target="_top">Contents</a> 
</font>
</p>
</td>   

</tr> 
</table>  

<p align="center">
<img src="../images/articles00.gif" width="500" height="16" alt="Articles">
</p>
 </div>

   <!-- Begin Article Header -->    

 <table border="0" cellpadding="0" cellspacing="0" width="100%"> 

 <colgroup>         

 <col width="6%">         

  <col width="94%"> </colgroup>           

 <tr>                 

   <td><img src="../images/spacer00.gif" width="10" height="10" alt="spacer"></td> 

                 <td>

 <h3 class="blue">D-Lib Magazine<br>May 2004</h3>  

                <h6 class="blue">Volume 10 Number 5<br><br>

            ISSN 1082-9873</h6>           

 

<H2 class="blue-space">Determining Space from Place for Natural History Collections</H2>
<H3 class="blue-space">In a Distributed Digital Library Environment</H3>


 </td>          

 </tr>           

 <tr>                   

<td>&nbsp;</td>                  

<td>

  

<p class="blue"> <a href="../authors/05authors.html#BEAMAN">Reed Beaman</a><br> 
Peabody Museum, Yale University<br>
&lt;reed.beaman&#064;yale.edu&gt;
 </p>

<p class="blue"> <a href="../authors/05authors.html#WIECZOREK">John Wieczorek</a><br> 
Museum of Vertebrate Zoology, University of California, Berkeley<br>
&lt;tuco&#064;socrates.berkeley.edu&gt;
 </p>


<p class="blue"> <a href="../authors/05authors.html#BLUM">Stan Blum</a><br> 
California Academy of Science<br>
&lt;sblum&#064;calacademy.org&gt;
 </p>


</td>          </tr>  </table>      <div class="center">

 <p><img src="../images/redline00.gif" width="500" height="2" alt="Red Line"></p> </div> 

  

   <table border="0" cellpadding="0" cellspacing="0" width="90%"> <colgroup>

         <col width="6%">          <col width="94%"> </colgroup>

           <tr>                  

<td><img src="../images/spacer00.gif" width="10" height="10" alt="spacer"></td>                 

<td>

  <!-- Abstract or TOC goes here --> 

<H3 class="blue">Abstract</H3>

<p class="blue">More than a billion biological specimens have been collected, preserved, and deposited in the permanent collections of museums and herbaria around the world. These specimens are the foundation of our knowledge about biological diversity, past and present. Researchers in biodiversity informatics are engaged in providing digital access to the basic biodiversity data associated with specimens, as well as new software tools and services that will create novel research opportunities for ecological analysis, predictive modeling, and synthesis. Greater access to structured biodiversity information also directly benefits applied areas of conservation and resource management. Digitizing the data associated with a billion specimens is an enormous task, and much of it still lies before us. Already, however, tens of millions of specimen records have been captured in collection management systems that represent a solid foundation for comprehensive digital libraries in the museum community. 
</p>

<p class="blue">
Of the various classes of information linked to biological specimens, geospatial coordinates used for mapping species distributions are among the most widely demanded by the scientific community and the general public. Providing these coordinates (geospatial referencing) has proven a significant challenge. Nearly every specimen is associated with a georeference, most often a textual representation of the place from whence it came, but few carry with them quantitative geospatial coordinates. We address here advances made in the task of geospatial referencing for biological collections.
</p>
<!-- Story goes next --> 


<H3>Background:  the case for collections</H3>

<p>
Managing finite natural resources is among the greatest challenges in this century facing biologists, information scientists, managers, economists, and policy-makers. Meeting this challenge is critical for sustaining human growth and prosperity, maintaining economic stability, and improving the quality of life for all species. Knowledge of biological diversity is fundamental to natural resource management, and the urgency for this knowledge ever increases as we convert the final tracts of the natural landscape into human-managed systems. Much of this knowledge remains locked in physical archives and on library shelves. In order for biodiversity management to keep pace with the rate of resource exploitation, digital access to biological diversity information is imperative in this decade.
</p>

<p>
Specimens, the primary objects in a biological collection, are of principal importance in documenting the spectacular biological diversity of plants and animals on this planet. Specimen data document the identities, habitats, histories, and spatial distributions of the roughly 1.75 million described species of life on earth, and provide the fundamental resource for identifying the estimated tens of millions of species that remain to be discovered and described (<a href="05beaman.html#Wilson">Wilson</a>, 2000). More than 250 years of natural history exploration have contributed to a vast worldwide collection of over one billion biological specimens. These specimens are essential pieces of the puzzle for answering four very basic questions: How many species are there?  What are their names?  How are they related to each other?  Where are they found?  Addressing these questions has been the domain of biological systematics (or taxonomy) since the time of Linnaeus (ca. 1735). Yet, to this day, answers to these four questions are largely incomplete. Biodiversity informatics facilitates the effort to resolve these questions, seeks to create new efficiencies for data capture and retrieval, and provides novel approaches for interdisciplinary analysis and visualization. 
</p>

<p>
The focal topic of this article is our access to digital data, particularly geospatial data, associated with biological collections. The digital acquisition, integration and application of biological collections data are increasingly viewed as fundamental to biodiversity research, education, and natural resource management. Many specimens linger for decades on the shelves of thousands of individual collections before a specialist gets an opportunity to study them. This deficit exists in large part because access to collections is limited by their distributed nature.
</p>


<h3>The ABCs of accessing biological collections</h3>

<p>
Providing digital access to biological collections is enabled by two critical technologies: (1) the digital capture of natural history collection data, and (2) the ongoing development of a cyberinfrastructure that links these digital collections into a global distributed digital library environment. A great deal of progress has been made recently on developing the infrastructure to provide digital access to data. Models for community solutions for distributed data access are evident in numerous projects such as the Global Biodiversity Information Facility (<a href="05beaman.html#GBIF">GBIF</a>, 2002), Australia's Virtual Herbarium (<a href="05beaman.html#AVH">AVH</a>, 2002), the Biological Collection Access Service for Europe (<a href="05beaman.html#BioCASE">BioCASE</a>, 2003), the Comisi	&#243;n Nacional para Conocimiento y Uso de la Biodiversidad (<a href="05beaman.html#CONABIO">CONABIO</a>, 2002), the European Natural History Specimen Information Network (<a href="05beaman.html#ENHSIN">ENHSIN</a>, 2000), <a href="05beaman.html#FishNet">FishNet</a> (2000), <a href="05beaman.html#HerpNet">HerpNET</a> (2003), and the Mammal Networked Information System (<a href="05beaman.html#MaNIS">MaNIS</a>, 2001), which are establishing distributed Internet-based information systems for accessing, integrating, and conducting predictive biotic modeling on specimen-based biodiversity data. Significant technological development, funding, and effort have gone into these projects, primarily for distributed information retrieval, data capture, and human-mediated geospatial referencing.
</p>


<h3>Space is the place</h3>

<p>
Every specimen has an associated core set of data that answers what, when, how, and where. A taxonomic name indicates the object's biological identity, the "collecting method" specifies how it was collected, and a list of one or more "collectors" indicates who collected it and when. While modern collectors are usually able to provide geospatial references in the form of coordinates derived from the Global Positioning System (GPS), the vast majority of specimens collected more than 10 years ago have only a natural language description, a textual georeference, as their specification of place.
</p>

<p>
Quantitative geospatial references are critical elements of biological collection data, because they enable us to integrate information within and beyond the biological domain. Geospatial referencing provides the means to link specimen data to the rapidly growing body of spatial environmental data for interdisciplinary research into complex phenomena. For example, access to geospatially referenced data from specimens provides a quantitative basis for biodiversity analyses and predictive niche modeling and for determining sampling densities of various sites (e.g., Figure 1): 
</p>

 
<p align="center" class="smallfontcell3"><img src="fig-1.jpg" width="495" height="672" alt="Image showing comparison of locality samples"><br><br>
<b>Figure 1. Comparison of localities sampled (A) with density
of specimens sampled (B). Density plots provide a more 
informative visualization of sampling completeness. 
Specimen data provided by Kristof Zyskowski, Peabody 
Museum.</b>
</p>


<p>
Because most collecting localities are in textual form, the museum community is facing an enormous task of converting natural language georeferences of place to geospatial references of space. The task of retrospectively deriving space from place, whether carried out using paper maps and rulers or with desktop GIS tools, is non-trivial when considered for all collections worldwide. This can represent an additional 25-100% overhead on the cost of digital data capture. A challenge therefore arises about what we can do to accelerate this process.
</p>

<p>
Locality information in biological datasets is by no means standardized, but it is to some extent similar across collections, making the task of parsing tractable. Similarities notwithstanding, there remain a number of interesting challenges. A few examples of textual locality statements illustrating these challenges are shown in Table 1.
</p>


<table border="1" width="90%" cellspacing="0" cellpadding="6" align="center">
	<tr>
		<td valign="top"><b>Example textual georeference</b></td>
		<td valign="top"><b>Challenge posed</b></td>
	</tr>
	<tr>
		<td valign="top">Wakarusa, 24 mi WSW of Lawrence</td>
		<td valign="top">Two or more locations described that are not exactly the same place</td>
	</tr>
	<tr>
		<td valign="top">Moccasin Creek on Hog Island</td>
		<td valign="top">Topological nesting</td>
	</tr>
	<tr>
		<td valign="top">Bupo [?Buso] River, 15 miles [24 km] E of Lae</td>
		<td valign="top">Complex grammar</td>
	</tr>
	<tr>
		<td valign="top">16 km (by road) N of Lawrence</td>
		<td valign="top">Linear feature measurement</td>
	</tr>
	<tr>
		<td valign="top">On the road between Sydney and Bathurst</td>
		<td valign="top">Linear ambiguity</td>
	</tr>
	<tr>
		<td valign="top">Southeast Michigan</td>
		<td valign="top">Vague localities</td>
	</tr>
	<tr>
		<td valign="top">Yugoslavia</td>
		<td valign="top">Political borders change over time</td>
	</tr>
	<tr>
		<td valign="top">British North Borneo</td>
		<td valign="top">Historical placename</td>
	</tr>
</table>


<p align="center">
<b>Table 1. Sample textual georeference and the challenges posed.</b>
</p>


<p>
We summarize below a semi-automated methodology based on collaboration and web services that are accessible to individual data providers and are interoperable with existing data networks. The solution encompasses natural language processing to interpret descriptive localities (geoparsing), placename matching to digital gazetteer data to register localities with known geographic coordinates, error analysis to self-document uncertainties in the resulting geographic descriptions, and data validation tools with which to visualize the results of geospatial references. This solution provides cost-effective added value through an economy of scale. Given the scope of both the specimen locality data and the demand for it in a readily usable form, efficiency and accuracy are of prime importance in our task of geospatial referencing.
</p>


<h3>Collaborative geospatial referencing</h3>

<p>
Individual institutions housing biological collections typically lack the resources or informatics expertise to meet the challenges of georeferencing alone. By designing an innovative collaborative geospatial referencing methodology, participants in the National Science Foundation funded Mammal Networked Information System (<a href="05beaman.html#MaNIS">MaNIS</a>, 2001) have achieved mean geospatial referencing rates of 16 localities per hour. Since MaNIS participants are referencing localities applicable to an average of about five specimens, the actual rate of specimen geospatial referencing is closer to 80 per hour (<a href="05beaman.html#Wieczorek-etal">Wieczorek <i>et al</i>.</a>, 2004). 
</p>

<p>
Users of biological collection data have differing needs for spatial resolution, implying that accuracy and precision information need to be captured as integral components of the georeferencing process. In addition, this information will be essential if biological collection data is to be integrated with other scalable spatial data (<a href="05beaman.html#Withey">Withey</a> <i>et al</i>., 2002). One of the key aspects of collaborative geospatial referencing for MaNIS is to provide an uncertainty measured as a maximum error radius around a geospatial reference point using a specialized <i>Georeferencing Calculator</i> (<a href="05beaman.html#Wieczorek-2001b">Wieczorek</a>, 2001b) implemented as a Java applet. Uncertainty values are calculated based on the characteristics of spatial relationships described in the natural language georeference, geospatial extent of a place, knowledge (or lack of knowledge) about geodetic datums, and levels of implied precision. The algorithms embedded in the <i>Georeferencing Calculator</i> are documented in the MaNIS Georeferencing Guidelines (<a href="05beaman.html#Wieczorek-2001a">Wieczorek</a>, 2001a). 
</p>

<p>
MaNIS data are currently available through a distributed network of providers accessible through one or more presentation layers or portals. This distributed network is implemented through a schema independent protocol, Distributed Generic Information Retrieval (<a href="05beaman.html#DiGIR">DiGIR</a>, 2001). DiGIR was developed in part through the effort of co-authors Wieczorek and Blum as an essential means of sharing collection data in the biological community.
</p>


<h3>Web services</h3>  

<p>
Digital data capture rates can and must be improved. While there are now tens of millions of digital data biological collection records available through the various sources mentioned above, it is a small percentage of the total. Development of automated and semi-automated geospatial referencing will greatly enhance the efficiency and efficacy of geographic data capture.
BioGeomancer (<a href="05beaman.html#Beaman">Beaman</a>, 2003) is a conversion engine for automating the conversion of references of place into geospatial references for natural history collections. The intent is to be able to handle language constructs such as those listed in Table 1. In its current incarnation, this BioGeomancer can parse English language placename descriptions and provide a set of latitude/longitude coordinates associated with that description. It provides offset calculations for when a collection is textually georeferenced at a given distance and cardinal direction from a named place.
</p>

<p>
The basic steps in the process include pre-processing text for language, locale or project specific anomalies (e.g., standardizing abbreviations) followed by one or more iterations of a phrase analysis algorithm. The description is segmented by punctuation, prepositions, and stop words into separate phrases and each phrase is analyzed independently. Within each of the phrase analysis iterations, text parsing and pattern matching using regular expressions involves detecting feature types (e.g., National Park, Island), placenames, and their inter-relationships. 
</p>

<p>
Candidate placenames are passed as a query to a local PostgreSQL database of gazetteer entries from the Geographic Names Information System (GNIS) (<a href="05beaman.html#USGS">USGS</a>, 1997) for U.S. data and National Geospatial-Intelligence Agency GEOnet Names Server (<a href="05beaman.html#NGA">NGA</a>, 2003) for all other countries. Geographic offsets (e.g., 2.5 km WNW of...) are detected and calculated relative to returned place names. 
The query interfaces allow either single or batch processing of multiple locality records. Web services are implemented through a Simple Object Access Protocol (<a href="05beaman.html#SOAP">SOAP</a>, 2000) interface for interoperability with other applications. Cross-platform interoperability has been tested with client applications written in various languages including Perl, PHP, Java, and .NET. For human-based access, a simple web form mediates as a wrapper for SOAP requests, post-processing responses with XSL stylesheets into HTML or passing the responses to an interactive web mapping interface.
</p>

<p>
A response summary and matched record detail are returned. Ambiguities in the response are caused by certain conditions, for example, the original designation of place includes more than one placename (e.g., "Micanopy, 11 miles S of Gainesville") and placename lookup in the gazetteers results in multiple places with the same name (the <i>homonym problem</i>). These ambiguities are handled by including an uncertainty estimate in the summary. This is calculated by placing a bounding box around all returned points, then measuring the distance from the centroid to a corner of the box. In a semi-automated environment, the error estimate can be used to automatically flag geospatial references that exceed a threshold for ambiguity.
</p>


<h3>Giving back: event gazetteers</h3>

<p>
Resolution of homonyms and synonyms in gazetteers is an area where knowledge derived from biological collections may contribute to gazetteer knowledge bases. For example, Madagascar presents an unusual homonymic challenge for geospatial referencing. There are 52 matched records for the place "Manakana" in Madagascar in the GeoNet Name Server. A bounding box placed around these matches has a distance of 1,368 km from corner to corner. Numerous synonymic variations in spelling also compound the ambiguity inherent in retrospective geospatial referencing. Further, numerous rural communities have moved in recent historical times and have retained the same name.
</p>

<p>
Thus, it will be important to describe these data beyond placenames and geographical feature types. Traditional gazetteer schemas generally define a placename, a feature type, coordinates, and some indication into which larger administrative entities a place fits. The Alexandria Digital Library (ADL) Gazetteer includes concepts of time-stamps (to address the issues of name, boundary, type, and relationship changes over time) and further descriptive information. We find that it will be extremely informative to extend specialized gazetteer schemas to include the concept of <i>collection events</i> (the name of a collector or expedition and a date of collection). As an example of how this might be useful, historical specimen records that we have checked often do not include higher order administrative units (e.g., states, provinces, counties, districts, etc.) but do include the collector's name and date of collection. By linking the "who, when, and where" of collecting activities, the development of <i>collection event gazetteers</i> can solve many of the problems of spatial homonymy. Homonyms can be distinguished by knowing the collectors' identities and location during a particular time period. These are data elements that are not only found on specimen label data, but also with notes, logs, diaries, and maps associated with collections.
</p>


<h3>Systematic transformations</h3>

<p>
Biological specimens are the primary source of information for measurements and observations used to infer relationships in the tree of life. Knowledge is also built on diverse additional resources in natural history collections, including illustrations, photographs, collectors' original notes, diaries, and maps. In the same way that medical informatics is able to provide distance-based diagnosis and treatment, access to biological collection data creates a virtual <i>collaboratory</i> for systematic study where information synthesis and assimilation that used to take days or weeks can occur in a few seconds. Providing this access to the scientific community facilitates systematic research and novel types of biogeographic analysis through GIS-based mapping, analysis, and visualization of species distributions. Indeed, these changes reach well beyond the systematics community. Until very recently, only specialists in systematics had access to this treasure trove of biological data. Improving access to biological collections in related disciplines adds tremendous value to the collections and our ability to synthesize biological data into associated domains. For example, scientists with improved access to species distribution data can improve the accuracy and precision of predictive distribution modeling for invasive species, pests, and disease-related research. Conservation and resource managers can better target areas of high biodiversity for protection and management. Students and educators can have access to the same information available to researchers scaled through appropriate views. 
</p>

<p>
The biodiversity community is by necessity embracing new technologies that transform how systematists conduct their science and how biological collections are managed. For example, an enormous transformation in systematic knowledge has occurred over the last few decades through the use of molecular (e.g., DNA sequencing) and computationally intensive methods of phylogenetic interpretation. This transformation has primarily affected our understanding of organismic relationships, amplifying our ability to count, classify, and name organisms, and has transpired in the laboratory at the molecular and algorithmic levels. Molecular systematics has greatly modified the scientific landscape for understanding our biological heritage. Still, there is the critical need to link molecular data to real organisms. Specimens serve as the vouchers for molecular biology, ecology, physiology, and pathology, representing the crucial comparative means for linking names to organisms. The practice of biology without vouchers creates an undocumented link between experimental organisms and scientific names. The names, distributions, and economic values are often the only aspects of most species that the lay community (including legislative bodies) will ever see.
</p>


<h3>Conclusions</h3>

<p>
Geospatial referencing is one of the new technologies that promises to unlock the biological and geographical knowledge contained in the world's natural history collections. It adds tremendous value to the use of biological specimens in the biological community and beyond, by allowing us to integrate this rich source of biological knowledge with geospatially referenced data in other domains through digital library architectures. Significant challenges remain ahead for improving technologies and methodologies that spatially enable natural history collections. As a foundation of knowledge for our planetary biological diversity, we must meet these challenges as a priority, and geospatial referencing is a prerequisite for the scientific synthesis essential for understanding and managing the complexities of our natural environment. 
</p>


<h3>References</h3>

<p>
<a name="AVH">AVH</a>. 2002. Australia's Virtual Herbarium, (<a href="http://www.anbg.gov.au/avh.html" target="_blank">http://www.anbg.gov.au/avh.html</a>).
</p>

<p>
<a name="Beaman">Beaman</a>, R. 2003. Biogeomancer: Automated georeferencing for natural history collections, (<a href="http://www.biogeomancer.org" target="_blank">http://www.biogeomancer.org</a>).
</p>

<p>
<a name="BioCASE">BioCASE</a>. 2003. Biological Collection Access Service for Europe, (<a href="http://www.biocase.org" target="_blank">http://www.biocase.org</a>). 
</p>



<p>
<a name="CONABIO">CONABIO</a>. 2002. Comisi&#243;n Nacional para Conocimiento y Uso de la Biodiversidad, (<a href="http://www.conabio.gob.mx/" target="_blank">http://www.conabio.gob.mx/</a>).
</p>

<p>
<a name="DiGIR">DiGIR</a>. 2001. Distributed Generic Information Retrieval, (<a href="http://digir.sourceforge.net" target="_blank">http://digir.sourceforge.net</a>).
</p>

<p>
<a name="ENHSIN">ENHSIN</a>. 2000. European Natural History Specimen Information Network, (<a href="http://www.nhm.ac.uk/science/rco/enhsin/" target="_blank">http://www.nhm.ac.uk/science/rco/enhsin/</a>). 
</p>

<p>
<a name="FishNet">FishNet</a>. 2000. The FishNet Distributed Biodiversity Information System, (<a href="http://habanero.nhm.ukans.edu/fishnet/" target="_blank">http://habanero.nhm.ukans.edu/fishnet/</a>). </p>


<p>
<a name="GBIF">GBIF</a>. 2002. The Global Biodiversity Information Facility, (<a href="http://www.gbif.org" target="_blank">http://www.gbif.org</a>). 
</p>


<p>
<a name="HerpNet">HerpNET</a>. 2003, (<a href="http://www.herpnet.org" target="_blank">http://www.herpnet.org</a>).
</p>



<p>
<a name="MaNIS">MaNIS</a>. 2001. The Mammal Networked Information System, (<a href="http://dlp.cs.berkeley.edu/manis" target="_blank">http://dlp.cs.berkeley.edu/manis</a>). 
</p>

<p>
<a name="NGA">NGA</a>. 2003. National Geospatial-Intelligence Agency GEONet Names Server, (<a href="http://earth-info.nga.mil/gns/html/index.html" target="_blank">http://earth-info.nga.mil/gns/html/index.html</a>).
</p>

<p>
<a name="SOAP">SOAP</a>. 2000. Simple Object Access Protocol (SOAP) 1.1, (<a href="http://www.w3.org/TR/SOAP/" target="_blank">http://www.w3.org/TR/SOAP/</a>). 
</p>

<p>
<a name="USGS">USGS</a>. 1997. Geographic Names Information System (GNIS), (<a href="http://gnis.usgs.gov/" target="_blank">http://gnis.usgs.gov/</a>). 
</p>

<p>
<a name="Wieczorek-2001a">Wieczorek</a>, J.R. 2001a. MaNIS: Georeferencing Guidelines, (<a href="http://elib.cs.berkeley.edu/manis/GeorefGuide.html" target="_blank">http://elib.cs.berkeley.edu/manis/GeorefGuide.html</a>)
</p>

<p>
<a name="Wieczorek-2001b">Wieczorek</a>, J.R. 2001b. Georeferencing Calculator, (<a href="http://elib.cs.berkeley.edu/manis/gc.html" target="_blank">http://elib.cs.berkeley.edu/manis/gc.html</a>). 
</p>

<p>
<a name="Wieczorek-etal">Wieczorek</a>, J., Guo, Q., and Hijmans, R. In review, 2004. The point-radius method for georeferencing locality descriptions and calculating associated uncertainty. <i>International Journal of Geographical Information Science</i>.
</p>

<p>
<a name="Wilson">Wilson</a>, E. O. 2000. On the future of conservation biology. <i>Conservation Biology</i>. Vol. 14(1):1-3. 
</p>

<p>
<a name="Withey">Withey</a>, A., W. Michener, and P. Tooby, (eds). 2002. <i>Scalable Information Networks for the Environment (SINE): Report of an NSF-sponsored workshop</i>, San Diego Supercomputer Center, October 29-31, 2001. 65 pp.
</p>




 <!-- Standard Copyright line here  -->
<center><h6>Copyright &copy; 2004 
Reed Beaman, John Wieczorek, and Stan Blum</h6>   
</center> 
</td>     

</tr>    

<!-- Begin the bottom sections -->      

<tr>      

 <td><img src="../images/spacer00.gif" width="10" height="10" alt="spacer"></td>

     <td> <hr width="80%" noshade size="1"></td>   

 </tr>    

 <tr>      

 <td><img src="../images/spacer00.gif" width="10" height="10" alt="spacer"></td>     <td>     

 <p class="cbs">     

 <a href="05beaman.html#Top">Top</a>      | <a href="../05contents.html">Contents</a><br>



      <a href="../../../Architext/AT-dlib2query.html">Search</a>      |  <a href="../../../author-index.html">Author Index</a>      |  <a href="../../../title-index.html">Title Index</a>      |  <a href="../../../back.html">Back Issues</a><br> 

<a href="../reid/05reid.html">Previous Article</a>      |  <a href="../05inbrief.html">In Brief</a> <br><a href="../../../dlib.html">Home</a>     | <a href="https://www.dlib.org/cdn-cgi/l/email-protection#06626a6f64466568746f28746375726968287067287375">E-mail the Editor</a></p>      </td>    </tr>     <tr>       <td><img src="../images/spacer00.gif" width="10" height="10" alt="spacer"></td>     <td> <hr width="80%" noshade size="1"></td>    </tr>    

 <tr>      

<td><img src="../images/spacer00.gif" width="10" height="10" alt="spacer"></td>



      <td>     

 <p class="small70"><a href="../../../access.html">D-Lib Magazine Access Terms and Conditions</a></p> 

      <p class="small70"><a href="https://www.doi.org"><b>DOI</b></a>:     10.1045/may2004-beaman</p> 

       <p> &nbsp;</p>  </td>

     </tr> 

   </table>  

  <script data-cfasync="false" src="../../../cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script></body>   

</html>

