<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" />
<meta name="DOI" content="10.1045/january2015-slaughter" />
<meta name="description" content="D-Lib Magazine" /> 
<meta name="keywords" content="Systematic Reviews, Clinical Guidelines, Evidence-based Medicine, Ontologies, Enhanced Publication, Semantic Web Standards" />
<link rel="metadata" href="01slaughter.meta.xml" />
<link rel="metadata" href="../01bib.meta.bib" />
<link rel="metadata" href="../01ris.meta.ris" />
<link href="../../../style/style1.css" rel="stylesheet" type="text/css" />
<title>Enabling Living Systematic Reviews and Clinical Guidelines through Semantic Technologies</title>
</head>

<body>
<form action="https://www.dlib.org/cgi-bin/search.cgi" method="get">

<table width="100%" border="0" cellpadding="0" cellspacing="0" bgcolor="#2b538e">
<tr>
<td><img src="../../../img2/space.gif" alt="" width="10" height="2" /></td></tr>
</table>

<table width="100%" border="0" cellpadding="0" cellspacing="0">
<tr>
<td valign="bottom" colspan="4" align="right" bgcolor="#4078b1">

<table border="0">
<tr>
<td align="right" class="search"><img src="../../../img2/search2.gif" alt="" width="51" height="20" align="middle" />Search D-Lib:</td>

<td>
<input type="text" name="words" value="" size="25" />
</td>

<td align="left" valign="middle">
<input type="submit" name="search" value="Go!" />
<input type="hidden" name="config" value="htdig" />
<input type="hidden" name="restrict" value="" />
<input type="hidden" name="exclude" value="" /> 
</td>
</tr>
</table>

</td></tr></table>

<table width="100%" border="0" cellpadding="0" cellspacing="0">
<tr>
<td valign="bottom" colspan="4">

<table width="100%" border="0" cellpadding="0" cellspacing="0" bgcolor="#e04c1e" id="outer" summary="Main Table">
<tr>
<td><img src="../../../img2/space.gif" alt="" width="10" height="1" /></td></tr>
</table>

<table width="100%" border="0" cellpadding="0" cellspacing="0" bgcolor="#F6F6F6" id="bannertable">
  <tr>
    <td width="830" bgcolor="#4078b1" class="backBannerImage" align="left"><img src="../../../img2/D-Lib-blocks.gif" alt="D-Lib Magazine" width="450" height="100" border="0" /></td>
  </tr>
  <tr>
    <td width="830" bgcolor="#e04c1e"><img src="../../../img2/transparent.gif" alt="spacer" height="1" /></td>
  </tr>
  <tr>
    <td width="830" bgcolor="#eda443" align="left"><img src="../../../img2/magazine.gif" alt="The Magazine of Digital Library Research" width="830" height="24" border="0" /></td>
  </tr>
  <tr>
    <td width="830" bgcolor="#e04c1e"><img src="../../../img2/transparent.gif" alt="spacer" height="1" /></td>
   </tr>
</table>

<table width="100%" border="0" cellpadding="0" cellspacing="0" id="navtable">
  <tr>
    <td width="5" height="20" bgcolor="#2b538e">&nbsp;</td>
    <td width="24" height="20" bgcolor="#2b538e"><img src="../../../img2/transparent.gif" alt="" width="24" height="20" /></td>
    <td height="20" align="left" bgcolor="#2b538e" class="navtext" nowrap="nowrap"><a href="../../../dlib.html">HOME</a>&nbsp;|&nbsp;<a href="../../../about.html">ABOUT D-LIB</a>&nbsp;|&nbsp;<a href="../../../contents.html" class="navtext">CURRENT ISSUE</a>&nbsp;|&nbsp;<a href="../../../back.html">ARCHIVE</a>&nbsp;|&nbsp;<a href="../../../author-index.html">INDEXES</a>&nbsp;|&nbsp;<a href="../../../groups.html">CALENDAR</a>&nbsp;|&nbsp;<a href="../../author-guidelines.html">AUTHOR GUIDELINES</a>&nbsp;|&nbsp;<a href="https://www.dlib.org/mailman/listinfo/dlib-subscribers">SUBSCRIBE</a>&nbsp;|&nbsp;<a href="../../letters.html">CONTACT D-LIB</a></td>
    <td width="5" height="20" bgcolor="#2b538e">&nbsp;</td>
  </tr>
</table>

<table width="100%" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td width="55" height="1" bgcolor="#e04c1e"><img src="../../../img2/space.gif" alt="transparent image" width="1" height="1" /></td></tr>
</table>

<!-- CONTENT TABLE -->
<table width="100%" border="0" align="center" cellpadding="0" cellspacing="0">
  <tr>
  <td>
 
<!-- BEGIN MAIN CONTENT TABLE -->

<table width="100%" border="0" cellspacing="0" cellpadding="10" bgcolor="#ffffff">
<tr>

<td width="10"><img src="../../../img2/space.gif" alt="" width="1" height="1" /></td>

<td valign="top"> 

<h3 class="blue-space">D-Lib Magazine</h3>
<p class="blue">January/February 2015<br />
Volume 21, Number 1/2<br />
<a href="../01contents.html">Table of Contents</a>
</p> 

<div class="divider-full">&nbsp;</div>

<h3 class="blue-space">Enabling Living Systematic Reviews and Clinical Guidelines through Semantic Technologies</h3>

<p class="blue">
Laura Slaughter<br />
The Interventional Centre, Oslo University Hospital (OUS), Norway<br /> 
laura.slaughter&#064;rr-research.no<br /><br />

Christopher Friis Berntsen<br />
Internal Medicine Department, Innlandet Hosptial Trust and MAGICorg, Norway<br />
christopherfriis.berntsen&#064;sykehuset-innlandet.no<br /><br />

Linn Brandt<br />
Internal Medicine Department, Innlandet Hosptial Trust and MAGICorg, Norway<br />
linn&#064;magicproject.org<br /><br />

Chris Mavergames<br />
Informatics and Knowledge Management Department, The Cochrane Collaboration, Germany<br />
cmavergames&#064;cochrane.org

<br /><br />DOI: 10.1045/january2015-slaughter
 </p>

<div class="divider-full">&nbsp;</div>

<p class="blue"><a href="01slaughter.print.html" class="fc">Printer-friendly Version</a></p>

<div class="divider-full">&nbsp;</div>

 <!-- Abstract or TOC goes here --> 

<h3 class="blue">Abstract</h3>

<p class="blue">
In clinical medicine, secondary research that produces systematic reviews and clinical practice guidelines is key to sound decision-making and quality care. Having machine-readable primary study publications, namely the methods and results of published human clinical trials can greatly improve the process of summarizing and synthesizing knowledge in medicine. In this short introduction to the problem, we provide a brief review of the related literature on various efforts to produce semantic technologies for sharing and reusing content from clinical investigations (RCTs and other clinical primary studies). Using an illustrative case, we outline some of the necessary metadata that needs to be captured in order to achieve some initial automation in authorship of systematic reviews and clinical guidelines. In addition, we list <i>desiderata</i> that we believe are needed to reduce the time and costs of maintaining these documents. These include linking provenance information to a much longer scientific investigation lifecycle, one that incorporates a single study's role all the way through its use in clinical guideline recommendations for patient treatments.
</p>

<p class="blue">Keywords: Systematic Reviews, Clinical Guidelines, Evidence-based Medicine, Ontologies, Enhanced Publication, Semantic Web Standards</p>

<!-- Article goes next --> 

<div class="divider-full">&nbsp;</div>
<h3>1 Introduction</h3>

<p>The importance of secondary research in clinical medicine cannot be ignored in discussions concerning digital content and machine-readable publications. Sound healthcare decision-making is dependent on efforts that bridge clinical research knowledge to recommendations about what actions to take in clinical practice. The main issue at hand is that the work of summarizing and synthesizing human clinical studies is a time-consuming and expensive process that can benefit from new semantic technologies and data sharing. Supporting this notion, Bechhofer <i>et al</i>. 2009 [<a href="01slaughter.html#1">1</a>] introduced the idea of Research Objects (RO) as a solution for exchanging reusable, machine-readable scientific publications. They wrote about this issue and and explained that "the information used to support clinical decisions is not dynamically linked to the cumulative knowledge of the best practice from research and audit." </p>

<p>Maintaining systematic reviews and clinical guidelines with the latest knowledge available involves having a notion of a "living publication" rather than a static document. This approach to disseminate knowledge in a format that is easily consumed and shared is compatible with using semantic web technologies, linked data and ontologies. The flow of knowledge from "downstream" (primary publications) to "upstream" (clinical guidelines and decision support systems) must be coordinated through innovative workflow tools. In this position paper, we address the needs for specific semantic technologies to realize the "living systematic review" proposed by Elliott <i>et al</i>., [<a href="01slaughter.html#2">2</a>] and the dynamic updating of guidelines proposed by Vandvik <i>et al</i>. [<a href="01slaughter.html#3">3</a>]. The concept of "living" refers to enhancing accuracy and utility of health evidence by developing persistent, dynamic digital publications that are easily adjustable and transportable. Unlike other conceptualizations of "living documents" [<a href="01slaughter.html#4">4</a>], the idea is not centered on the traditional paper-based structuring of scientific reports. It concerns the dissemination of the living data and information (i.e. the meta-analysis) that forms the basis of formally written frozen documents. </p>

<p>In this position paper, we have created a list of <i>desiderata</i> &#151; our view of the necessary next steps towards development of new "living systematic reviews and clinical guideline" technologies. Firstly, we present a short introduction to the process of authoring/maintaining systematic reviews and clinical guidelines. We then briefly review the most substantial efforts and communities we have identified, including more general initiatives such as the open Research Objects community and nanopublication as well as the specifically biomedical "health informatics" centered work. As part of our evaluation process, we used an illustrative case to find relevant classes and concepts in available biomedical ontologies. We then go into more detail as we discuss the <i>desiderata</i>. We argue that biomedical and life sciences efforts towards realizing the goal of machine-readable scientific publications need to also consider the extended life of primary studies, specifically for secondary research as well as the flow of knowledge to other information systems in medicine such as clinical decision-making systems. We discuss the needs for "threading" [<a href="01slaughter.html#5">5</a>] data and information across publications. Although it has been proposed that the entire workflow to produce these publications can be automatized [<a href="01slaughter.html#6">6</a>], there is still a need for human involvement. For instance, humans must be involved in assessing the quality of primary studies to avoid bias in the meta-analyses or to author guideline recommendations about what clinical actions/treatments to take in specific circumstances. Current systems are essentially "workflow support tools" and the <i>desiderata</i> reflect necessary semantic technology contributions to improve these types of tools. </p>

<div class="divider-dot">&nbsp;</div>
<h4>1.1 Authoring and Maintaining Systematic Reviews and Clinical Practice Guidelines</h4>

<p><b><i>1.1.1 Translating Research Evidence into Clinical Practice Recommendations</i></b></p>

<p>As advocates of evidence-based medicine (EBM) have professed for more than two decades, clinicians should strive to base their decisions on the best available evidence [<a href="01slaughter.html#7">7</a>]&#151;[<a href="01slaughter.html#9">9</a>]. To answer questions about the relative effectiveness of treatment and prevention interventions, compiling systematic reviews from a series of well-conducted randomized-controlled trials (RCTs) is the preferred methodology for synthesizing such a body of evidence [<a href="01slaughter.html#10">10</a>]. Meta-analyses may, if applied appropriately, strengthen an inference about the efficacy of a treatment or prevention intervention, due to the added statistical power yielded by the pooling of results from individual studies [<a href="01slaughter.html#11">11</a>]. For questions about prognosis and prevalence issues, observational study designs such as cohort or cross-sectional studies may be apt, and may form the basis of systematic reviews. </p>

<p>A widespread way of translating research evidence into actionable treatment recommendations is the development of clinical practice guidelines (CPGs) [<a href="01slaughter.html#9">9</a>], [<a href="01slaughter.html#12">12</a>]. CPG recommendations are ideally informed by existing or ad hoc systematic reviews [<a href="01slaughter.html#13">13</a>], or in their absence, by single RCTs, observational studies, or at times physiological studies or random clinical observations, in descending order of methodological strength [<a href="01slaughter.html#10">10</a>].</p>

<p>During the last one and a half decades, the process of systematically appraising evidence in the process of developing treatment recommendations has received much attention &#151; notably through the GRADE working group [<a href="01slaughter.html#14">14</a>], [<a href="01slaughter.html#15">15</a>] and the work of the Cochrane Collaboration [<a href="01slaughter.html#16">16</a>]. Importantly, treatment recommendations may vary across clinical and geographical settings due to differences in therapeutic traditions, epidemiological situations or availability of interventions [<a href="01slaughter.html#17">17</a>]. Possible differences in patient preferences should also be considered when issuing treatment recommendations, especially when the presumed benefit of a treatment balances closely with potential negative consequences for the patient [<a href="01slaughter.html#15">15</a>].</p>

<p><i><b>1.1.2 Evidence Retrieval Process</b></i></p>

<p>Although the same summarized evidence may give rise to different recommendations in different settings, a thorough process of evidence retrieval is nevertheless at the core of the evidence synthesis process, whether its end-point is a systematic review or a CPG recommendation. This remains a highly labor-intensive process, involving extensive literature searches in a range of databases. For a typical systematic review of the effect of a therapeutical intervention, this would involve a search for reports of clinical trials at the minimum in PubMed, Embase and the Cochrane Central Registry of Randomized Trials (CENTRAL), and possibly more specialized databases, depending on the topic [<a href="01slaughter.html#18">18</a>].</p>

<p>A basic search in these databases would typically consist of a Boolean OR clause combining all known synonyms, possible spellings as well as database-specific controlled vocabulary terms for the relevant population, which again would be combined with a similar clause for the relevant therapeutical intervention in an AND clause [18]. Optionally, this may be combined in a similar fashion with terms for the comparators and outcomes of interest, although this is generally considered to lower the sensitivity of the search, and not applicable for systematic reviews [<a href="01slaughter.html#18">18</a>], [<a href="01slaughter.html#19">19</a>]. This search approach is commonly referred to as a PICO search, an acronym for population &#151; intervention &#151; comparator &#151; outcome, which is a common way to phrase a precise clinical question [<a href="01slaughter.html#10">10</a>], [<a href="01slaughter.html#20">20</a>]. The PICO elements reflect the design of a clinical trial, where a population defined by certain inclusion and exclusion criteria is allocated into two (or more) groups (the intervention and comparator groups), and outcomes of interest measured independently for the groups.</p>

<p>An inherent problem of the above-mentioned search approach is that the free-text search strings applied may not be reflected in the title or abstract sections. Terms for the population and intervention may be hidden inside non-searchable text (such as the article PDFs resting on the individual journal providers' sites, outside the searchable databases) [<a href="01slaughter.html#20">20</a>]. As the indexing process in the above-mentioned study databases is at least partly manual, articles are also prone to be over- or under-indexed with controlled vocabulary terms due to inter-indexer inconsistency [<a href="01slaughter.html#21">21</a>]. Current automated algorithms for indexing articles are also imperfect [<a href="01slaughter.html#22">22</a>].</p>

<p>Manual screening of large, unfocused search returns to identify the articles actually relevant to a review question or clinical question in a CPG may mean browsing hundreds to thousands of citations. Indeed, this has been identified as a significant obstacle to the timely updating of CPGs [<a href="01slaughter.html#23">23</a>]&#151;[<a href="01slaughter.html#25">25</a>]. Although numerous efforts have been made to develop search filters retrieving relevant studies with greater precision [<a href="01slaughter.html#26">26</a>]&#151;[<a href="01slaughter.html#31">31</a>], this inevitably involves a trade-off between controlling the amount of noise in the search output and the risk of missing a relevant study due to the properties of the filter. Other efforts to ease the process of retrieving relevant, methodologically sound studies include the PLUS database maintained by McMaster University, where screeners manually pre-appraise studies, including only studies fitting certain methodological criteria [<a href="01slaughter.html#32">32</a>].</p>

<p><b><i>1.1.3 Evidence Appraisal</i></b></p>

<p>Whether for creating a CPG recommendation or for performing systematic reviews or meta-analyses, a thorough knowledge of the methodological strengths and weaknesses of a study is needed to make an inference about the generalizability of its findings [<a href="01slaughter.html#10">10</a>], [<a href="01slaughter.html#15">15</a>]. Lacking certain qualities in the study design, like randomization, may make the study prone to <i>bias</i>, thus weakening its predictive value or even leading to wrong conclusions altogether [<a href="01slaughter.html#35">35</a>]. Erroneous applications of statistical methods or missing data could also lead to invalid conclusions [<a href="01slaughter.html#33">33</a>]. For systematic reviews, meta-analyses and CPGs, the validity of conclusions or recommendations is no better than the studies that they are based upon.</p>

<p>After the screening process, information about studies that are deemed relevant is generally extracted manually using data extraction forms [<a href="01slaughter.html#16">16</a>]. At the very least, to even consider including a study in an evidence summary, a reviewer or CPG developer would need to know some basic meta-data about it. The study population and interventions of the study must match the ones of the review or clinical question [<a href="01slaughter.html#34">34</a>], and the design of the study must be stated clearly to allow a judgment about its methodological quality.</p>

<p>For the analyses itself, the reviewers need to be able to extract information about results including outcome data, computed estimates, statistical uncertainties, as well as the statistical methods applied. Ideally the reviewers would also have access to the raw data collected in the studies, to be able to verify the calculations done and to pool data across studies. Judgments about whether to finally include or exclude studies from reviews and meta-analyses are complex, and should be done in a systematic way according to pre-specified criteria, and in duplicate [<a href="01slaughter.html#34">34</a>]. </p>

<p><b><i>1.1.4 Current Practices for Keeping Systematic Reviews and Clinical Practice Guidelines Updated</i></b></p>

<p>Partly due to the burden of literature monitoring, practices for updating CPGs and systematic reviews vary greatly. Some groups advocate updating at fixed intervals [<a href="01slaughter.html#35">35</a>], others having tried to identify literature-monitoring strategies on smaller scales to identify studies with certain properties ("signal studies") that could possibly impact the conclusions of a review in the static form it was last published [<a href="01slaughter.html#36">36</a>]. Yet other groups aim towards <i>living guidelines</i> [<a href="01slaughter.html#3">3</a>], [<a href="01slaughter.html#24">24</a>] or <i>living systematic reviews</i> [<a href="01slaughter.html#2">2</a>] that are updated dynamically as new evidence is identified.</p>

<div class="divider-dot">&nbsp;</div>
<h4>1.2 Background</h4>

<p>The Human Studies Database Project (HSDP) is part of more extensive work that has been on-going for many years in an effort to create the infrastructure needed for institutions to share the design (protocols) of clinical studies. Sim <i>et al</i>. [<a href="01slaughter.html#37">37</a>] write about the efforts of the project, which is a multi-institutional collaboration. They postulate that federated searching over locally controlled databases is the most feasible data sharing approach. It is based on this assumption that they propose the Ontology for Clinical Research (OCRe) as the common model defining the concepts that will be queried across the various local databases [<a href="01slaughter.html#20">20</a>]. This ontology attempts to model eligibility criteria, study design types, study outcomes and variables, scientific query (hypothesis), and analysis in general. OCRe is formalized in OWL 2. OCRe is not only a typology for annotating study data, but it also uses OWL logical axioms for reasoning to provide decision support for critiquing studies, specifically to determine risk of bias. The current work is focused on deploying the OCRe in a workflow support system and study results repository, the AHRQ's Systematic Review Data Repository (SRDR) as well as extending the model of OCRe to include clinical study results data. They are also working towards data sharing between the SRDR and Cochrane Collaboration's Central Register of Studies. </p>

<p>One of the main limitations of the OCRe and HSDP is that it lacks some of the relevant classes that are important for supporting the evidence appraisal process described above, and maintaining systematic reviews and CPGs. There also appears to be a "gap" in needed reasoning about statistical methods. Another drawback is the need for evaluation of the ontology in practical applications (e.g. real-life systems for authoring protocols) and adoption by developers of tools for dissemination/reuse of this data. </p>

<p>There have been various proposals for linking external data, such as from health records, to the hypotheses, methods, and/or results from clinical studies. Sim <i>et al</i>. [<a href="01slaughter.html#20">20</a>] has, in addition, discussed the lifecycle of the clinical study in relation to the "learning health system" concept [<a href="01slaughter.html#38">38</a>]. In a learning health system, data and information extracted from care documentation (the Electronic Health Record), is reused as a basis for improving quality of care and in forming a basis for clinical trials. Connections can be made between "study and clinical phenotypes that can be matched and integrated in the process of care". </p>

<p>The Clinical Data Interchange Standards Consortium's (CDISC) Object Data Model (ODM) is a format for the interchange and exchange of clinical research study data and metadata. The ODM model includes metadata that describes study setup, operation/procedures, and analysis. There is a recent initiative to convert the ODM to RDF format, which has resulted in a set of schemas that are to be used in various projects, such as EHR4CR [<a href="01slaughter.html#57">57</a>]. These projects such as EHR4CR as well as another called i2b2 have used ODM for interoperability between information systems. Their main goal is to be able to support physicians conducting clinical studies using data from electronic health records. Meineke <i>et al</i>. [<a href="01slaughter.html#58">58</a>] and Ganslandt <i>et al</i>. [<a href="01slaughter.html#59">59</a>] discuss these goals as well as the process of building support tools such as databases for annotated clinical data from health records that are then the basis for clinical studies. The ODM is interesting as a candidate for metadata associated with primary clinical studies that could be used when authoring systematic reviews and guidelines. </p>

<p>A much earlier paper by Chalmers and Altman in 1999 [<a href="01slaughter.html#5">5</a>] introduced the concept of the "threaded electronic publication" to highlight discrepancies between clinical protocols as they are documented for ethics committees from the reporting of the protocol procedures as written in published methodology sections of articles. The <a href="http://crossref.org/crossmark/">CrossMark</a> tool  was developed to demonstrate the "threading" described in the Altmann and Chalmers paper, "linking a trials registration record and all articles reporting the protocol or findings of the trial, published in different journals or by different publishers in a single thread" [<a href="01slaughter.html#39">39</a>].</p>

<p>The Biotea Project [<a href="01slaughter.html#40">40</a>] has created a repository of annotated PubMed Central articles as RDF triples, making SPARQL searches of the full-text content a possibility. In conjunction with Bio2RDF and bioportal ontologies, this group has already explored and tested search interfaces with semantic browsing of content [<a href="01slaughter.html#41">41</a>]&#151;[<a href="01slaughter.html#43">43</a>]. Scientific publications are seen as an interface to the Web of Data [<a href="01slaughter.html#42">42</a>]. </p>

<p>The open <a href="http://www.researchobject.org">Research Objects (RO)</a> community  has focused mainly on the life sciences, where a growing number of life scientists are "calling for models and tools that can be used to organize, package and share research investigations". Belhajjame <i>et al</i>. 2014 [<a href="01slaughter.html#44">44</a>] describes a suite of ontologies that represent study methods and data, and to some extent the results reported. Research Objects are containers that hold various investigation related resources that are aggregated in one sharable unit. They provide contextual information related to the experiments/studies, and stress that scientific thinking must be captured as the study progresses, dynamically and iteratively through different hypotheses and designs. The ontologies used by ROs include the Open Archive Initiative Object Exchange and Reuse (OAI-ORE) model, annotation ontologies, the W3C standard <a href="http://www.w3.org/TR/prov-o">Provenance Ontology (PROV-O)</a>, and the Research Object Core Ontology that is based on ORE but has additional aggregation structures. The use of ROs for primary clinical studies is not discussed in the literature currently, even though the problem of maintaining clinical knowledge was presented in an earlier publication introducing the content [<a href="01slaughter.html#1">1</a>]. The RO Digital Library and RODL interfaces are centered on life sciences work, but these may be reused and linked to further clinical knowledge if the approach is widely adopted in human clinical studies. </p>

<p>Nanopublications [<a href="01slaughter.html#45">45</a>], [<a href="01slaughter.html#46">46</a>] were proposed as a means for specifying key scientific statements or facts that occur in publications, such as "malaria is transmitted by mosquitos". These statements are represented as RDF triples and contain additional annotation statements that contain authorship information and place the nanopublication in the context of the longer, full publication (provenance metadata). Examples of nanopublication are found in the Huntington's Disease case study [<a href="01slaughter.html#47">47</a>], [<a href="01slaughter.html#48">48</a>] and contain statements that enables life scientists to expose the results of their analyses as scientific assertions. This use of nanopublication is relevant since it involves asserting results of studies, a needed component to realizing living systematic reviews and clinical guidelines. </p>

<div class="divider-full">&nbsp;</div>
<h3>2 Illustrative Case</h3>

<p>In addition to the ontologies related to projects presented in the background section, there are a number of relevant biomedical ontologies that can be useful and reusable resources. In keeping with the idea of reuse that is stressed in the RO and nanopublications communities, we began to sketch an illustrative case for evaluative purposes. We look at the data needed for the evidence synthesis and appraisal process, and discuss representation of the extracted elements from a single primary study. Currently, during the authoring of systematic reviews and guidelines, authors carry out their work manually. A human being opens a PDF file, reads, and locates text for copy/paste from the primary study into a spreadsheet for further use in reviews or meta-analyses. </p>

<p>For our working example, we have chosen a study that we have found to be well-conducted in terms of methodological rigor, providing extensive information on study design and statistical methods applied. The study by Kahn <i>et al</i>. [<a href="01slaughter.html#49">49</a>], on the efficacy of compression stockings for preventing the development of post-thrombotic syndrome in patients having suffered a lower-extremity deep vein thrombosis, was recently published in a high-impact medical journal. We chose this study first and foremost because of the quite complete data presented in the paper. We have extracted the results and metadata from the paper, and entered it into a structured, machine-readable format using publicly available ontologies. This illustrative case shows some of the tedious nature of the work to extract metadata from primary studies.</p>
 
<p>We do provide some RDF code in the discussion. The publication Kahn <i>et al.</i> [<a href="01slaughter.html#49">49</a>] is represented as <span class="codelg">:Pub-Kahn14</span>. We skip the RDF representation of the bibliographic information to save space for the issues related to machine-readable metadata for the study design and analysis. The representation of the study data is not exhaustive, but we intend to provide examples from each of the PICO domains.</p>

<p>The key types of information that are necessary to determine whether a primary study is relevant to a systematic review or guideline, are its PICO elements: <i>population</i>, <i>intervention</i>, <i>comparison</i>, and <i>outcome</i>. Overall classes are found in the PICO ontology [<a href="01slaughter.html#50">50</a>], however this work is still in the development stage, so we have sought out additional ontologies that might be useable.  </p>

<p>P (population) is a class in a number of biomedical ontologies. <span class="codelg">Eligible population</span> is found in the OCRe and is defined as "The collection of screened organisms that satisfies all eligibility criteria for a study." They have also defined a class for <span class="codelg">Study population</span> as well as various classes for subpopulation. <span class="codelg">Population</span> is also found in the <a href="http://obi-ontology.org/page/Main_Page">Ontology for Biomedical Investigation (OBI)</a> and the <a href="http://bioportal.bioontology.org/ontologies/ACGT-MO">ACGT master ontology</a>. We find <span class="codelg">Group assignment</span> as a class that is a process in the <a href="https://code.google.com/p/obcs/">Ontology for Biological and Clinical Statistics</a>, and is the process of assigning individuals into specific roles. </p>

<p>Knowing the population is essential, but also, authors of systematic reviews and guidelines need to know the inclusion and exclusion criteria of the study. Inclusion criteria and exclusion criteria are factors that determine whether a patient can be assigned to a study group, if they are eligible or not eligible. These are often medical conditions or specific demographic characteristics, as well as reasons that would prohibit participation. In our example, one inclusion criterion is a <i>medical condition</i>. In Kahn <i>et al.</i>, an eligible individual must have a deep vein thrombosis (DVT), the body location of the DVT is "in the popliteal or more proximal deep leg veins", and this needs to be confirmed with a diagnostic test, a confirmation using ultrasound within the time boundary of the past 14 days. OCRe, as stated above, contains <span class="codelg">Eligible population</span>, but we could not identify classes to explicitly represent inclusion/exclusion. This current representation of study population in the OCRe is not very granular, only allowing a free-text label to be attached. In our view this will need to be augmented to achieve the goals we outline above for searchability, at least allowing the association of controlled vocabulary terms with the population definition. This would also ideally be supplemented by the possibility of defining exclusion and inclusion criteria. The semantics of study eligibility criteria are complex, as have been discussed separately in a paper by Ross <i>et al</i>. [<a href="01slaughter.html#51">51</a>] but we could argue that only a minor amount of structured vocabulary identifying the population as such might increase search precision. In this case, the <a href="http://www.nlm.nih.gov/pubs/factsheets/mesh.html">MeSH</a> or <a href="http://www.nlm.nih.gov/research/umls/Snomed/snomed_main.html">SNOMED</a> code for deep vein thrombosis might suffice.</p>

<p>We were however able to identify these relevant classes in the ACGT master ontology: <span class="codelg">EligibilityCriterion</span>, <span class="codelg">ExclusionCriterion</span>, and <span class="codelg">InclusionCriterion</span>. In this case, the most important machine-readable information that needs to be conveyed is:&nbsp;<span class="codelg">:Pub-Kahn-Population hasInclusionCriteria :DeepVeinThrombosis</span>, and the entire representation would include an assertion that connects deep vein thrombosis to being a criterion for inclusion in the study. We would also include coding from <a href="http://www.who.int/classifications/icd/en/">ICD</a>, SNOMED or MeSH for the "deep vein thrombosis" concept.  A more advanced representation would indicate body location, which for this study has to be "in the popliteal or more proximal deep leg veins" and the diagnostic confirmation required for eligibility "ultrasound confirmed". The ability to represent this information is more difficult since it requires more complex reified statements. </p>

<p>For the I (intervention) and C (comparison) from PICO, the comparison is often made against a placebo, as in our illustrative case. The study intervention is "active 30&#151;40 mmHg graduated elastic compression stockings (ECS), exchanged every 6 months" and is compared with "placebo ECS with identical appearance but less than 5 mmHg compression at the ankle". The PICO ontology has these classes available, and we could include metadata using the MeSH coding of the intervention: </p>

<p class="indentLeft"><span class="codelg">mesh:D053828 a pico:Intervention .</span></p>

<p>Outcomes can be divided into what are determined to be "primary outcomes" and "secondary outcomes". For the Kahn <i>et al</i>. [<a href="01slaughter.html#49">49</a>] study, the primary outcome is presence of "post-thrombotic syndrome" (PTS). Other outcome information is a mixed-bag of related concepts that includes the analysis methods, various clinical scales/diagnostic tools used to determine presence of a primary outcome medical condition (like PTS), adverse events, and quality of life. Some necessary classes could be found scattered across various ontologies, including, for example, the <a href="http://www.obofoundry.org/cgi-bin/detail.cgi?id=AERO">adverse events reporting ontology</a>. In the following example, <span class="codelg">HazardRatio</span> is a statistical outcome measure used to express the relative chance of event-free survival during the study period, and GinsbergScale is a diagnostic tool. </p>

<p class="indentLeft"><span class="codelg">:PTS has_outcome_measure :HazardRatio<br />
:PTS is_assigned_outcome :PrimaryOutcome<br />
:PTS has_snomed_code snomed:C20283957<br />
:PTS has_assessment_method :GinsbergScale</span></p>

<p>The classes in OCRe ontology let us characterize the study design as a double-blinded randomized-controlled trial. We might have added more characteristics, such as it being a multi-centre study, but will limit this example for the sake of space constraints:</p>

<p class="indentLeft"><span class="codelg">@prefix ocre:&lt;http://purl.org/net/OCRe/OCRe.owl/&gt;</span></p>

<p class="indentLeft"><span class="codelg">:Pub-Kahn14 a ocre:Study ;<br />
hasDesign ocre:ParallelGroupStudyDesign ;br />
hasBlinding ocre:SubjectBlinded ;br />
hasBlinding ocre:InvestigatorBlinded ;br />
hasAllocationScheme ocre:BlockRandomization ;br />
hasRandomizationScheme ocre:CentralRandomization .</span></p>

<p>Finally, we outline the dependent (outcome) variables and statistical methods used for arriving at the final summary statistics. This is not an exhaustive description of the above-mentioned study, as several baseline variables (covariates) and secondary outcomes were described in the Kahn <i>et al</i>. paper. For simplicity, we limit this example to the primary outcome, whereas more outcomes and greater detail could easily be added.</p>

<p class="indentLeft"><span class="codelg">:PostThromboticSyndrome a ocre:studyPhenomenon .<br />
:PostThromboticSyndrome a ocre:CD .<br />  
:PostThromboticSyndrome ocre:hasConceptId "D054070" .<br />
:GinsbergScale a ocre:AssessmentMethodSpecification .<br />
:GinsbergScale ocre:hasDescription "The Ginsberg PTS scoring scale is..." .<br />
:PrimaryAnalysis a ocre:CoxRegression .</span></p>

<p>What we found was that even though ontologies contain some necessary classes and concepts, there is not one that is currently satisfactory to use by itself as a standard to represent study data needed for evidence extraction and secondary synthesis. The ontologies we have identified are somewhat lacking on entities suitable for the reporting of final results, and to varying extent suitable for describing the statistical methods applied for arriving at these. Coincidentally, the Cox regression method used in the study by Kahn <i>et al</i>. to derive a hazard ratio is not represented in the <a href="http://isa-tools.github.io/stato/">STATO ontology</a>, although STATO generally describes a broader range of statistical methods. Classes suitable for representing aggregate statistics such as odds ratios, risk ratios, hazard ratios or mean differences from clinical studies are not usual (although the Ontology of Biological and Clinical Statistics has a '<span class="codelg">relative risk</span>' class). Such outcome data are mostly simple literals. Easy identification of these result data would relieve data extraction for secondary use. </p>

<div class="divider-full">&nbsp;</div>
<h3>3 Desiderata</h3>

<p>The <i>living systematic</i> review and the <i>living clinical guideline</i> are currently in a state of nascent development. Institutions that are producing and storing this secondary publication of clinical knowledge are aware of the many benefits of using semantic technologies and linked data. We have created a list of <i>desiderata</i> that will serve as a "roadmap" and a set of topics for further investigation.</p>

<div class="divider-dot">&nbsp;</div>
<h4>3.1 Ontology Development</h4>

<p>Like the RO ontology suite described for the life sciences [<a href="01slaughter.html#44">44</a>], enabling needed environments for sharing and reuse of clinical human studies will require additional ontology resources that are not currently available. </p>

<p><i>A PICO Ontology</i></p>

<p>The proposed "PICO ontology" [<a href="01slaughter.html#50">50</a>], being developed by Cochrane, who are pioneers in systematic reviews and evidence synthesis, represents study hypotheses and clinical questions in the standardized form of Population, Intervention, Comparator, and Outcome(s). The PICO ontology in development from Cochrane will be an effort to represent classes not covered by OCRe. It also will be the basis for "threading" from systematic reviews to clinical guideline recommendations, as well as to the underlying study report data, comparisons and analyses. It will also allow connections to study data in centralized registers such as <a href="https://clinicaltrials.gov">clinicaltrials.gov</a>. These "threads" are essential. </p>

<p><i>"Results" and an "Analysis" Ontologies</i></p>

<p>An ontology for representing results of clinical studies needs to be developed, and in addition, a "statistical-methods-analysis" ontology that includes "risk of bias" models. As discussed above in previous sections of the paper, the results of primary studies should be sharable and interoperable as well as the data and methods. A "statistical methods-analysis + risk-of-bias ontology", or simply an "analysis ontology", covers the analysis of the data and assists with this need to address bias, i.e. that just crunching data into an "automated review" [<a href="01slaughter.html#6">6</a>] is not reasonable without application of human assessment for methodological reasons. Neither of these ontologies exists in an implementable form, but there have been related projects that provide a foundation. </p>

<p>Ontologies that have modeled aspects of results of clinical studies cover two areas specifically in medicine: cancer and neurosurgery. They were built for project demonstrations of semantic technology and model a subset of patient data and results. Previous work to design a system for managing clinical trial data has taken place in an earlier EU project that produced the ObTiMA system "ontology-based management of clinical trials" [<a href="01slaughter.html#52">52</a>].This system is composed of two components, one for setting up clinical trials and the other for handling patient data during trials. Both components depend on an ontology (the <a href="http://bioportal.bioontology.org/ontologies/ACGT-MO">ACGT master ontology</a>) [<a href="01slaughter.html#53">53</a>] that covers the research spectrum specifically in the area of cancer trials. Zaveri <i>et al.</i> [<a href="01slaughter.html#54">54</a>] built an ontology for reporting diagnostic study results in neurosurgery. Their <a href="http://bioportal.bioontology.org/ontologies/DIAGONT">computational diagnostic ontology</a> contains classes that are required to conduct SR-MA (Systematic Review &#151; Meta Analysis) for diagnostic studies that assist in standardized reporting of non-RCT diagnostic study publications. </p>

<div class="divider-dot">&nbsp;</div>
<h4>3.2 A Much Longer "Lifecycle" in an Ecosystem</h4>

<p>We have observed that many of the definitions of "publication lifecycle" seen across all the literature on machine-readable scientific publications have been too limited. For our own needs, we propose to map out the current lifecycle of clinical study publications, and extend this lifecycle to the entire evidence ecosystem. This includes demonstrating how elements from the study protocol are carried through "threaded publications" [<a href="01slaughter.html#39">39</a>], which include results and their role in secondary publications like the systematic review and meta-analyses, add connections to clinical guidelines decision support systems, and relationships to clinical practice documentation. In addition, provenance needs to be linked throughout the extended lifecycle, using the recently created PROVO or PROV ontology that "provides a set of classes, properties, and restrictions that can be used to represent and interchange provenance information generated in different systems and under different contexts".</p>

<p>The work on Research Objects has had a great deal of input from the life sciences. Ontologies used in Research Objects, such as the Ontology of Biomedical Investigation (OBI) can be extended as a model allowing reuse of data by clinical research with traditional clinical data types in combination with the use of genomic technologies. A related project that might provide some resources for infrastructure is i2b2, "Informatics for Integrating Biology and the Bedside" [<a href="01slaughter.html#55">55</a>]. </p>

<p>We propose nanopublication throughout the lifecycle. We need to identify where nanopublications will be most productive. This leads to further discussion of both credit as well as <i>accountability</i>. For instance, one means to provide incentive for keeping guidelines up to date is to give credit to the authors writing recommendations by allowing them to nanopublish them individually. </p>

<div class="divider-dot">&nbsp;</div>
<h4>3.3 Intelligent Workflow Support for Secondary Research and Improved Search Tools </h4>

<p><i><b>3.3.1 Minimum Contribution of the PICO, Results and Analysis Ontologies in the Process of Systematic Review and Clinical Practice Guideline Development</b></i></p>

<p>Even a rudimentary registration of metadata about a study report in a searchable database may improve the performance of the evidence-retrieval process. We propose that at the time of publication of a clinical trial report, machine-readable study properties using suitable ontologies that model PICO, statistics and methods/analysis, and results can be registered in the living systematic review workflow system, making it possible to do more precise searching and browsing of content related to the review.</p>

<p>Given the amount of resources medical journals spend on reviewing, proofreading and reformatting manuscripts before publication, we propose that they include registration of a limited amount of study data and metadata in a structured manner. If this was done using a standardized tool it would not necessarily add to the workload of the publication process, as the authors of the report, being familiar with the contents of their work, could perform parts of the this registration. The quality control of these data could and should then become an integral part of the proofreading and peer review process. </p>

<p>Journals may decide individually about the amount of content they are willing to share in freely searchable databases. If they are willing to make structured information about the PICO elements, study design/methods, and results available in the same fashion that abstracts presently are searchable, the precision of the evidence retrieval could improve significantly. The process of second-hand indexing of content would be bypassed, and the noise produced by wide free-text searches for PICO terms in title and abstract fields would diminish. The publication of the structured data and metadata could effectively be nanopublished or wrapped in a Research Object container. Likewise, a PICO and methods/analysis ontology could be applied to secondary medical literature, such as systematic reviews and meta-analyses, guidelines, editorials or comments where applicable.</p>

<p>In any case, a structured registration of clinical trial results including computed measures of statistical uncertainty, information of data processing, statistical methods and study design at the time of publication of a study report would significantly ease the data extraction process of systematic review and clinical practice guideline production. As outlined above, this may not necessarily add substantially to the workload of the publication process. </p>

<p><i><b>3.3.2 The Need for Tools to Manage Linked Clinical Study Data</b></i></p>

<p>Having tools to work with the clinical linked data publications will promote adoption and widespread use as it does in <a href="http://isacommons.org">ISA commons</a> used by life scientists. We propose development of tools, such as an enhanced "clinical trials management and reporting system" that will assist clinical trial planners in writing protocols and report data in a structured fashion, helping them to contribute to a wider community instead of working in isolation. Search tools for federated search over linked data repositories should be developed. Sim <i>et al.</i> [<a href="01slaughter.html#20">20</a>] state that it would be better to improve search with "interactive visualizations of the scientific structure of human studies like the tools that biomedical researchers have for visual exploration and query of gene sequences, pathways, and protein structures" We agree that linking the various clinical knowledge to other related sources requires search tools as well as visual analytics. This has been attempted by some groups, for instance, Saleem et al. 2010 [<a href="01slaughter.html#56">56</a>] created a user interface for integrating PubMed publications with the Linked Cancer Genome Atlas dataset. </p>

<div class="divider-full">&nbsp;</div>
<h3>4 Limitations</h3>

<p>We have not completely considered how the body of medical literature might be searched differently when we have access to more precise metadata and RDF triples as machine-readable publications. Widespread adoption of the PICO and methods ontologies in the future will greatly improve search functionality. However, even if only limited adoption of suggested semantic web technologies occurs, sharing structured extracted data across databases may partially allow the backlog of medical literature to be more efficiently synthesized and will speed up the process of authoring. Efforts to share such data are currently made through the <a href="http://www.cochrane.org">Cochrane Collaboration</a> and the <a href="https://srdr.ahrq.gov">US Systematic Review Data Repository (SRDR), Agency for Healthcare Research and Quality</a>.</p>

<div class="divider-full">&nbsp;</div>
<h3>5 Conclusions</h3>

<p>Secondary research, involving summarizing and synthesizing existing studies, is essential to clinical medicine. The need for basing patient care and treatment decisions on current medical knowledge drives the production of two types of documents that are key: systematic reviews and clinical practice guidelines. Systematic reviews cover primary research studies that are protocol-based, RCT studies for clinical questions, while the practice guidelines generally summaries these clinical questions into documents that provide recommendations for practitioners to follow on a certain condition or care topic. The authoring process and maintenance of these documents is both costly and labor intensive. We have therefore looked towards related communities for "enhanced publication", "executable papers", "nano-publications", "research objects" in addition to the healthcare informatics projects that are relevant, to find the "missing elements" and topics for further discussion. There are already ontologies that are applicable to certain phases of the evidence-to-recommendation translation process, notably the OCRe, but more work is needed especially on metadata and models for study analysis and results. The development of new tools for reporting on and searching for structured data from clinical trials are also required to implement these technologies in practice.</p>

<div class="divider-full">&nbsp;</div>
<h3>References</h3>

<p><a name="1">[1]</a> S. Bechhofer, J. Ainsworth, J. Bhagat, I. Buchan, P. Couch, D. Cruickshank, M. Delderfield, I. Dunlop, M. Gamble, C. Goble, D. Michaelides, P. Missier, S. Owen, D. Newman, D. De Roure, and S. Sufi, "Why Linked Data is Not Enough for Scientists," presented at the Sixth IEEE e&#151;Science conference (e-Science 2010), 2010. <a href="https://doi.org/10.1016/j.future.2011.08.004">http://doi.org/10.1016/j.future.2011.08.004</a></p>

<p><a name="2">[2]</a> J. H. Elliott, T. Turner, O. Clavisi, J. Thomas, J. P. T. Higgins, C. Mavergames, and R. L. Gruen, "Living Systematic Reviews: An Emerging Opportunity to Narrow the Evidence-Practice Gap," <i>PLoS Med</i>, vol. 11, no. 2, p. e1001603, Feb. 2014. <a href="https://doi.org/10.1371/journal.pmed.1001603">http://doi.org/10.1371/journal.pmed.1001603</a></p>

<p><a name="3">[3]</a> P. O. Vandvik, L. Brandt, P. Alonso-Coello, S. Treweek, E. A. Akl, A. Kristiansen, A. Fog-Heen, T. Agoritsas, V. M. Montori, and G. Guyatt, "Creating clinical practice guidelines we can trust, use, and share: a new era is imminent," <i>Chest</i>, vol. 144, no. 2, pp. 381&#151;389, Aug. 2013. <a href="https://doi.org/10.1378/chest.13-0746">http://doi.org/10.1378/chest.13-0746</a></p>

<p><a name="4">[4]</a> A. G. Castro, L. J. Garc&#237;a-Castro, A. Labarga, O. Giraldo, C. Monta&#241;a, K. O'Neil, and J. A. Bateman, "Annotating Atomic Components of Papers in Digital Libraries: The Semantic and Social Web Heading towards a Living Document Supporting eSciences," in <i>Database and Expert Systems Applications</i>, S. S. Bhowmick, J. K&#252;ng, and R. Wagner, Eds. Springer Berlin Heidelberg, 2009, pp. 287&#151;301. <a href="https://doi.org/10.1007/978-3-642-03573-9_24">http://doi.org/10.1007/978-3-642-03573-9_24</a></p>

<p><a name="5">[5]</a> I. Chalmers and D. G. Altman, "How can medical journals help prevent poor medical research? Some opportunities presented by electronic publishing," <i>The Lancet</i>, vol. 353, no. 9151, pp. 490&#151;493, Feb. 1999. <a href="https://doi.org/10.1016/S0140-6736(98)07618-1">http://doi.org/10.1016/S0140-6736(98)07618-1</a></p>

<p><a name="6">[6]</a> G. Tsafnat, A. Dunn, P. Glasziou, and E. Coiera, "The automation of systematic reviews," <i>BMJ</i>, vol. 346, no. Jan. 2010 1, pp. f139&#151;f139, Jan. 2013. <a href="https://doi.org/10.1136/bmj.f139">http://doi.org/10.1136/bmj.f139</a></p>

<p><a name="7">[7]</a> Evidence-Based Medicine Working Group, "Evidence-based medicine. A new approach to teaching the practice of medicine," <i>JAMA J. Am. Med. Assoc</i>., vol. 268, no. 17, pp. 2420&#151;2425, Nov. 1992. <a href="https://doi.org/10.1001/jama.1992.03490170092032">http://doi.org/10.1001/jama.1992.03490170092032</a></p>

<p><a name="8">[8]</a> S. E. Straus, W. S. Richardson, P. Glasziou, and R. B. Haynes, <i>Evidence-based medicine. How to practice and teach EBM</i>. Edinburgh: Elsevier Churchill Livingstone, 2011. ISBN-10: 0702031275.</p>

<p><a name="9">[9]</a> M. B. Harrison, F. Legare, I. D. Graham, and B. Fervers, "Adapting clinical practice guidelines to local context and assessing barriers to their use," <i>Cmaj</i>, vol. 182, pp. E78&#151;84, Feb. 2010. <a href="https://doi.org/10.1503/cmaj.081232">http://doi.org/10.1503/cmaj.081232</a></p>

<p><a name="10">[10]</a> G. Guyatt, D. Rennie, M. O. Meade, and D. J. Cook, <i>Users' guide to the medical literature. Essentials of evidence-based clinical practice</i>. New York: Jama Archives and Journals/McGraw Hill Medical, 2008. ISBN-10: 0071590382.</p>

<p><a name="11">[11]</a> J. J. Deeks, J. P. T. Higgins, and D. G. Altman, "Chapter 9: Analysing data and undertaking meta-analyses.," in <i><a href="http://handbook.cochrane.org">Cochrane Handbook for Systematic Reviews of Interventions Version 5.1.0 (updated March 2011)</a></i>., G. S. Higgins JPT, Ed. The Cochrane Collaboration, 2011.</p>

<p><a name="12">[12]</a> A. Qaseem, F. Forland, F. Macbeth, G. Ollenschlager, S. Phillips, and P. van der Wees, "Guidelines International Network: toward international standards for clinical practice guidelines," <i>Ann Intern Med</i>, vol. 156, pp. 525&#151;31, Apr. 2012. <a href="https://doi.org/10.7326/0003-4819-156-7-201204030-00009">http://doi.org/10.7326/0003-4819-156-7-201204030-00009</a></p>

<p><a name="13">[13]</a> R. Graham, M. Mancher, D. M. Wolman, S. Greenfield, and E. Steinberg, <i><a href="http://www.nap.edu/openbook.php?record_id=13058">Clinical practice guidelines we can trust</a></i>. National Academies Press, 2011.</p>

<p><a name="14">[14]</a> G. H. Guyatt, A. D. Oxman, H. J. Schunemann, P. Tugwell, and A. Knottnerus, "GRADE guidelines: a new series of articles in the Journal of Clinical Epidemiology," <i>J Clin Epidemiol.</i>, vol. 64, pp. 380&#151;2, Apr. 2011. <a href="https://doi.org/10.1016/j.jclinepi.2010.09.011">http://doi.org/10.1016/j.jclinepi.2010.09.011</a></p>

<p><a name="15">[15]</a> G. H. Guyatt, A. D. Oxman, G. E. Vist, R. Kunz, Y. Falck-Ytter, P. Alonso-Coello, and H. J. Schunemann, "GRADE: an emerging consensus on rating quality of evidence and strength of recommendations," <i>Bmj</i>, vol. 336, pp. 924&#151;6, Apr. 2008. <a href="https://doi.org/10.1136/bmj.39489.470347.AD">http://doi.org/10.1136/bmj.39489.470347.AD</a></p>

<p><a name="16">[16]</a> J. P. T. Higgins and J. J. Deeks, "Chapter 7: Selecting studies and collecting data," in <i><a href="http://handbook.cochrane.org">Cochrane Handbook for Systematic Reviews of Interventions Version 5.1.0 (updated March 2011)</a></i>, J. P. T. Higgins and S. Green, Eds. The Cochrane Collaboration, 2011.</p>

<p><a name="17">[17]</a> B. Fervers, J. S. Burgers, M. C. Haugh, J. Latreille, N. Mlika-Cabanne, L. Paquet, M. Coulombe, M. Poirier, and B. Burnand, "Adaptation of clinical guidelines: literature review and proposition for a framework and procedure," <i>Int J Qual Health Care</i>, vol. 18, pp. 167&#151;76, Jun. 2006. <a href="https://doi.org/10.1093/intqhc/mzi108">http://doi.org/10.1093/intqhc/mzi108</a></p>

<p><a name="18">[18]</a> C. Lefebvre, E. Manheimer, and J. Glanville, "Chapter 6: Searching for studies." in <i><a href="http://handbook.cochrane.org">Cochrane Handbook for Systematic Reviews of Interventions Version 5.1.0 (updated March 2011)</a></i>, G. S. Higgins JPT, Ed. The Cochrane Collaboration, 2011.</p>

<p><a name="19">[19]</a> T. Agoritsas, A. Merglen, D. S. Courvoisier, C. Combescure, N. Garin, A. Perrier, and T. V. Perneger, "Sensitivity and predictive value of 15 PubMed search strategies to answer clinical questions rated against full systematic reviews," <i>J Med Internet Res</i>, vol. 14, p. e85, 2012. <a href="https://doi.org/10.2196/jmir.2021">http://doi.org/10.2196/jmir.2021</a></p>

<p><a name="20">[20]</a> I. Sim, S. W. Tu, S. Carini, H. P. Lehmann, B. H. Pollock, M. Peleg, and K. M. Wittkowski, "The Ontology of Clinical Research (OCRe): An informatics foundation for the science of clinical research," <i>J Biomed Inf</i>., Nov. 2013. <a href="https://doi.org/10.1016/j.jbi.2013.11.002">http://doi.org/10.1016/j.jbi.2013.11.002</a></p>

<p><a name="21">[21]</a> M. E. Funk and C. A. Reid, "Indexing consistency in MEDLINE," <i>Bull Med Libr Assoc</i>, vol. 71, pp. 176&#151;83, Apr. 1983. PMCID: PMC227138.</p>

<p><a name="22">[22]</a> A. J. Yepes, J. G. Mork, D. Demner-Fushman, and A. R. Aronson, "Comparison and combination of several MeSH indexing approaches" <i>AMIA Annu Symp Proc</i>, vol. 2013, pp. 709&#151;18, 2013. PMCID: PMC3900212.</p>

<p><a name="23">[23]</a> E. Clark, E. F. Donovan, and P. Schoettker, "From outdated to updated, keeping clinical guidelines valid," <i>Int J Qual Health Care</i>, vol. 18, pp. 165&#151;6, June 2006. <a href="https://doi.org/10.1093/intqhc/mzl007">http://doi.org/10.1093/intqhc/mzl007</a></p>

<p><a name="24">[24]</a> P. Alonso-Coello, L. Martinez Garcia, J. M. Carrasco, I. Sola, S. Qureshi, and J. S. Burgers, "The updating of clinical practice guidelines: insights from an international survey," <i>Implement Sci</i>, vol. 6, p. 107, 2011. <a href="https://doi.org/10.1186/1748-5908-6-107">http://doi.org/10.1186/1748-5908-6-107</a></p>

<p><a name="25">[25]</a> L. Martinez Garcia, I. Arevalo-Rodriguez, I. Sola, R. B. Haynes, P. O. Vandvik, and P. Alonso-Coello, "Strategies for monitoring and updating clinical practice guidelines: a systematic review," <i>Implement Sci</i>, vol. 7, p. 109, 2012. <a href="https://doi.org/10.1186/1748-5908-7-109">http://doi.org/10.1186/1748-5908-7-109</a></p>

<p><a name="26">[26]</a> R. B. Haynes and N. Wilczynski, "Finding the gold in MEDLINE: clinical queries," <i>ACP J Club</i>, vol. 142, pp. A8&#151;9, Feb. 2005. <a href="https://doi.org/10.1136/ebm.10.4.101">http://doi.org/10.1136/ebm.10.4.101</a></p>

<p><a name="27">[27]</a> N. L. Wilczynski and R. B. Haynes, "Response to Corrao <i>et al</i>.,: Improving efficacy of PubMed clinical queries for retrieving scientifically strong studies on treatment," <i>J Am Med Inf. Assoc</i>, vol. 14, pp. 247&#151;8, Apr. 2007. <a href="https://doi.org/10.1197/jamia.M2297">http://doi.org/10.1197/jamia.M2297</a></p>

<p><a name="28">[28]</a> K. A. McKibbon, N. L. Wilczynski, and R. B. Haynes, "Retrieving randomized controlled trials from medline: a comparison of 38 published search filters," <i>Health Info Libr J</i>, vol. 26, pp. 187&#151;202, Sep. 2009. <a href="https://doi.org/10.1111/j.1471-1842.2008.00827.x">http://doi.org/10.1111/j.1471-1842.2008.00827.x</a></p>

<p><a name="29">[29]</a> 	C. Lokker, R. B. Haynes, N. L. Wilczynski, K. A. McKibbon, and S. D. Walter, "Retrieval of diagnostic and treatment studies for clinical use through PubMed and PubMed's Clinical Queries filters," <i>J Am Med Inf. Assoc</i>, vol. 18, pp. 652&#151;9, Oct. 2011. 
</p>

<p><a name="30">[30]</a> N. L. Wilczynski, K. A. McKibbon, and R. B. Haynes, "Sensitive Clinical Queries retrieved relevant systematic reviews as well as primary studies: an analytic survey," <i>J Clin Epidemiol</i>, vol. 64, pp. 1341&#151;9, Dec. 2011. <a href="https://doi.org/10.1016/j.jclinepi.2011.04.007">http://doi.org/10.1016/j.jclinepi.2011.04.007</a></p>

<p><a name="31">[31]</a> N. L. Wilczynski, K. A. McKibbon, S. D. Walter, A. X. Garg, and R. B. Haynes, "MEDLINE clinical queries are robust when searching in recent publishing years," <i>J Am Med Inf. Assoc</i>, vol. 20, pp. 363&#151;8, Apr. 2013. <a href="https://doi.org/10.1136/amiajnl-2012-001075">http://doi.org/10.1136/amiajnl-2012-001075</a></p>

<p><a name="32">[32]</a> B. J. Hemens and R. B. Haynes, "McMaster Premium Literature Service (PLUS) performed well for identifying new studies for updated Cochrane reviews,"<i> J Clin Epidemiol</i>, vol. 65, pp. 62&#151;72.e1, Jan. 2012. <a href="https://doi.org/10.1016/j.jclinepi.2011.02.010">http://doi.org/10.1016/j.jclinepi.2011.02.010</a></p>

<p><a name="33">[33]</a> J. P. T. Higgins, D. G. Altman, and J. A. C. Sterne, "Chapter 8: Assessing risk of bias in included studies," in <i><a href="http://handbook.cochrane.org">Cochrane Handbook for Systematic Reviews of Interventions Version 5.1.0 (updated March 2011)</a></i>, J. P. T. Higgins and S. Green, Eds. The Cochrane Collaboration, 2011. </p>

<p><a name="34">[34]</a> D. O'Connor, S. Green, and J. P. T. Higgins, "Chapter 5: Defining the review question and developing criteria for including studies," in <i><a href="http://handbook.cochrane.org">Cochrane Handbook for Systematic Reviews of Interventions Version 5.1.0 (updated March 2011)</a></i>, J. P. T. Higgins and S. Green, Eds. The Cochrane Collaboration, 2011.</p>

<p><a name="35">[35]</a> P. Shekelle, M. P. Eccles, J. M. Grimshaw, and S. H. Woolf, "When should clinical guidelines be updated?," <i>Bmj</i>, vol. 323, pp. 155&#151;7, Jul. 2001. <a href="https://doi.org/10.1136/bmj.323.7305.155">http://doi.org/10.1136/bmj.323.7305.155</a></p>

<p><a name="36">[36]</a> N. Ahmadzai, S. J. Newberry, M. A. Maglione, A. Tsertsvadze, M. T. Ansari, S. Hempel, A. Motala, S. Tsouros, J. J. Schneider Chafen, R. Shanman, D. Moher, and P. G. Shekelle, "A surveillance system to assess the need for updating systematic reviews," <i>Syst Rev</i>, vol. 2, p. 104, 2013. <a href="https://doi.org/10.1186/2046-4053-2-104">http://doi.org/10.1186/2046-4053-2-104</a></p>

<p><a name="37">[37]</a> I. Sim, S. Carini, S. Tu, R. Wynden, B. H. Pollock, S. A. Mollah, D. Gabriel, H. K. Hagler, R. H. Scheuermann, H. P. Lehmann, K. M. Wittkowski, M. Nahm, and S. Bakken, "The Human Studies Database Project: Federating Human Studies Design Data Using the Ontology of Clinical Research," <i>AMIA Summits Transl. Sci. Proc</i>., vol. 2010, pp. 51&#151;55, Mar. 2010. <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3041546/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3041546/</a></p>

<p><a name="38">[38]</a> C. P. Friedman, A. K. Wong, and D. Blumenthal, "<a href="https://www.networkforphl.org/_asset/3pcp7d/AchievingNationwideLHS.pdf">Achieving a nationwide learning health system</a>," <i>Sci. Transl. Med.</i>, vol. 2, no. 57, p. 57cm29, Nov. 2010. </p>

<p><a name="39">[39]</a> D. Shanahan, "<a href="http://blogs.biomedcentral.com/bmcblog/2014/01/31/threaded-publications-one-step-closer/">Threaded Publications: one step closer</a>," <i>BioMed Central blog</i>, 31-Jan-2014.</p>

<p><a name="40">[40]</a> L. J. G. Castro, C. McLaughlin, and A. Garcia, "Biotea: RDFizing PubMed Central in support for the paper as an interface to the Web of Data," <i>J. Biomed. Semant</i>., vol. 4, no. Suppl 1, p. S5, Apr. 2013. <a href="https://doi.org/10.1186/2041-1480-4-S1-S5">http://doi.org/10.1186/2041-1480-4-S1-S5</a></p>

<p><a name="41">[41]</a> L. J. Garc&#237;a Castro, O. X. Giraldo, and A. Garcia, "<a href="http://ceur-ws.org/Vol-903/paper-02.pdf">Using annotations to model discourse: an extension to the Annotation Ontology</a>," pp. 13&#151;22.</p>

<p><a name="42">[42]</a> L. J. Garc&#237;a-Castro, A. G. Castro, and J. G&#243;mez, "<a href="http://ceur-ws.org/Vol-952/paper_40.pdf">Conceptual Exploration of Documents and Digital Libraries in the Biomedical Domain</a>," in <i>Proceedings of the 5th International Workshop on Semantic Web Applications and Tools for Life Sciences</i>, Paris, France, November 28-30, 2012, 2012, vol. 952. </p>

<p><a name="43">[43]</a> L. J. Garc&#237;a-Castro, R. B. Llavori, D. Rebholz-Schuhmann, and A. G. Castro, "<a href="http://ceur-ws.org/Vol-994/paper-05.pdf">Connections across Scientific Publications based on Semantic Annotations</a>," in <i>Proceedings of the 3rd Workshop on Semantic Publishing</i>, Montpellier, France, May 26th, 2013, vol. 994, pp. 51&#151;62. </p>

<p><a name="44">[44]</a> K. Belhajjame, J. Zhao, D. Garijo, K. Hettne, R. Palma, &#211;. Corcho, J.-M. G&#243;mez-P&#233;rez, S. Bechhofer, G. Klyne, and C. Goble, "<a href="http://arxiv.org/abs/1401.4307">The Research Object Suite of Ontologies: Sharing and Exchanging Research Data and Methods on the Open Web</a>," <i>ArXiv14014307 Cs,</i> Jan. 2014. </p>

<p><a name="45">[45]</a> P. Groth, A. Gibson, and J. Velterop, "The anatomy of a nanopublication," <i>Inf Serv. Use</i>, vol. 30, no. 1&#151;2, pp. 51&#151;56, 2010. <a href="https://doi.org/10.3233/ISU-2010-0613">http://doi.org/10.3233/ISU-2010-0613</a></p>

<p><a name="46">[46]</a> P. Sernadela, E. van der Horst, M. Thompson, P. Lopes, M. Roos, and J. L. Oliveira, "A Nanopublishing Architecture for Biomedical Data," in <i>8th International Conference on Practical Applications of Computational Biology &amp; Bioinformatics (PACBB 2014)</i>, J. Saez-Rodriguez, M. P. Rocha, F. Fdez-Riverola, and J. F. D. P. Santana, Eds. Springer International Publishing, 2014, pp. 277&#151;284. <a href="https://doi.org/10.1007/978-3-319-07581-5_33">http://doi.org/10.1007/978-3-319-07581-5_33</a></p>

<p><a name="47">[47]</a> E. Mina, M. Thompson, R. Kaliyaperumal, J. Zhao, K. M. Hettne, E. Schultes, and M. Roos, "<a href="https://www.openphacts.org/documents/publications/Mina_SWAT4LS_Nanopublications%20for%20exposing%20experimental%20data%20in%20the%20life%20sciences_a%20Huntingtons%20Disease%20case%20study.pdf">Nanopublications for Exposing Experimental Data in the Life-sciences: A Huntington's Disease Case Study</a>," in <i>SWAT4LS</i>, 2013.</p>

<p><a name="48">[48]</a> S. Magliacane and P. T. Groth, "Towards Reconstructing the Provenance of Clinical Guidelines," in <i>Proceedings of the 5th International Workshop on Semantic Web Applications and Tools for Life Sciences</i>, Paris, France, November 28-30, 2012, 2012, vol. 952. <a href="http://ceur-ws.org/Vol-952/paper_36.pdf">http://ceur-ws.org/Vol-952/paper_36.pdf</a></p>

<p><a name="49">[49]</a> S. R. Kahn, S. Shapiro, P. S. Wells, M. A. Rodger, M. J. Kovacs, D. R. Anderson, V. Tagalakis, A. H. Houweling, T. Ducruet, C. Holcroft, M. Johri, S. Solymoss, M. J. Miron, E. Yeo, R. Smith, S. Schulman, J. Kassis, C. Kearon, I. Chagnon, T. Wong, C. Demers, R. Hanmiah, S. Kaatz, R. Selby, S. Rathbun, S. Desmarais, L. Opatrny, T. L. Ortel, and J. S. Ginsberg, "Compression stockings to prevent post-thrombotic syndrome: a randomised placebo-controlled trial," <i>Lancet</i>, vol. 383, no. 9920, pp. 880&#151;8, 2014. <a href="https://doi.org/10.1016/S0140-6736(13)61902-9">http://doi.org/10.1016/S0140-6736(13)61902-9</a></p>

<p><a name="50">[50]</a> C. Mavergames, S. Oliver, and L. Becker, "<a href="http://ceur-ws.org/Vol-994/paper-02.pdf">Systematic Reviews as an Interface to the Web of (Trial) Data: using PICO as an Ontology for Knowledge Synthesis in Evidence-based Healthcare Research</a>," in <i>Proceedings of the 3rd Workshop on Semantic Publishing</i>, Montpellier, France, May 26th, 2013, 2013, vol. 994, pp. 22&#151;26. </p>

<p><a name="51">[51]</a> J. Ross, S. Tu, S. Carini, and I. Sim, "<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3041539/">Analysis of Eligibility Criteria Complexity in Clinical Trials</a>," <i>AMIA Summits Transl. Sci. Proc</i>., vol. 2010, pp. 46&#151;50, Mar. 2010. </p>

<p><a name="52">[52]</a> H. Stenzhorn, G. Weiler, M. Brochhausen, F. Schera, V. Kritsotakis, M. Tsiknakis, S. Kiefer, and N. Graf, "The ObTiMA system &#151; ontology-based managing of clinical trials," <i>Stud. Health Technol. Infor</i>m., vol. 160, no. Pt 2, pp. 1090&#151;1094, 2010. doi: 10.3233/978-1-60750-588-4-1090<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3041539/"> http://doi.org/10.3233/978-1-60750-588-4-1090</a></p>

<p><a name="53">[53]</a> M. Brochhausen, A. D. Spear, C. Cocos, G. Weiler, L. Mart&#237;n, A. Anguita, H. Stenzhorn, E. Daskalaki, F. Schera, U. Schwarz, S. Sfakianakis, S. Kiefer, M. D&#246;rr, N. Graf, and M. Tsiknakis, "The ACGT Master Ontology and its applications&#151;towards an ontology-driven cancer research and management system," <i>J. Biomed. Inform</i>., vol. 44, no. 1, pp. 8&#151;25, Feb. 2011. <a href="https://doi.org/10.1016/j.jbi.2010.04.008">http://doi.org/10.1016/j.jbi.2010.04.008</a></p>

<p><a name="54">[54]</a> A. Zaveri, J. Shah, S. Pradhan, C. Rodrigues, J. Barros, B. T. Ang, and R. Pietrobon, "Center of Excellence in Research Reporting in Neurosurgery &#151; Diagnostic Ontology," <i>PLoS ONE</i>, vol. 7, no. 5, p. e36759, May 2012. <a href="https://doi.org/10.1371/journal.pone.0036759">http://doi.org/10.1371/journal.pone.0036759</a></p>

<p><a name="55">[55]</a> E. K. Johnson, S. Broder-Fingert, P. Tanpowpong, J. Bickel, J. R. Lightdale, and C. P. Nelson, "Use of the i2b2 research query tool to conduct a matched case-control clinical research study: advantages, disadvantages and methodological considerations," <i>BMC Med. Res. Methodol</i>., vol. 14, p. 16, 2014. <a href="https://doi.org/10.1186/1471-2288-14-16">http://doi.org/10.1186/1471-2288-14-16</a></p>

<p><a name="56">[56]</a> M. Saleem, "<a href="https://www.academia.edu/6743534/Big_Linked_Cancer_Data_Integrating_Linked_TCGA_and_PubMed">Big Linked Cancer Data: Integrating Linked TCGA and PubMed</a>."</p>

<p><a name="57">[57]</a> J. Doods,  F. Botteri, M. Dugas, F. Fritz, and EHR4CR WP7, "A European Inventory of Common Electronic Health Record Data Elements for Clinical Trial Feasibility." <i>Trials</i> 15 (2014): 18. <a href="https://doi.org/10.1186/1745-6215-15-18">http://doi.org/10.1186/1745-6215-15-18</a></p>

<p><a name="58">[58]</a> F. A. Meineke, S. St&#228;ubert, M. L&#246;be, and A. Winter, "A Comprehensive Clinical Research Database Based on CDISC ODM and i2b2." <i>Studies in Health Technology and Informatics</i> 205 (2014): 1115&#151;19. <a href="https://doi.org/10.3233/978-1-61499-432-9-1115">http://doi.org/10.3233/978-1-61499-432-9-1115</a></p>

<p><a name="59">[59]</a> T. Ganslandt, S. Mate, K. Helbing, U. Sax, and H. U. Prokosch, "Unlocking Data for Clinical Research &#151; The German i2b2 Experience." <i>Applied Clinical Informatics</i> 2, no. 1 (2011): 116&#151;27. <a href="https://doi.org/10.4338/ACI-2010-09-CR-0051">http://doi.org/10.4338/ACI-2010-09-CR-0051</a></p>

<div class="divider-full">&nbsp;</div>

<h3>About the Authors</h3>

<table border="0"  cellpadding="6" bgcolor="#FFFFFF"> 
<tr>
<td align="center"><img src="slaughter.png" class="border" alt="slaughter" width="100" height="128" /></td>
<td>
<p class="blue"><b>Laura Slaughter</b> (PhD) received both her MLS and PhD from the College of Information Studies at the University of Maryland, College Park. She defended her dissertation in 2002 on the topic of "Semantic Relationships in Health Consumer Questions and Physicians' Answers: A Basis for Representing Medical Knowledge and for Concept Exploration Interfaces. " She spent two years at Columbia University as a postdoc in the Dept of Biomedical Informatics. She is currently employed at Oslo University Hospital, Oslo, Norway. Her research expertise is within healthcare informatics, and covers areas including: knowledge representation, controlled vocabulary &amp; ontologies, semantic web, intelligent information systems, and clinical decision support. </p>
</td>
</tr>
</table>

<div class="divider-full">&nbsp;</div>

<table border="0"  cellpadding="6" bgcolor="#FFFFFF"> 
<tr>
<td align="center"><img src="berntsen.png" class="border" alt="berntsen" width="100" height="117" /></td>
<td>
<p class="blue"><b>Christopher Friis Berntsen</b> (MD, BA) is a hospital physician specializing in internal medicine at Lovisenberg Hospital in Oslo, Norway. He is also part-time employed as a researcher at Sykehuset Innlandet Hospital Trust in Gj&#248;vik, Norway, through the MAGIC (Making GRADE the Irresistible Choice) research and innovation program, focusing on the optimization of clinical practice guideline updating processes and evidence retrieval strategies.</p>
</td>
</tr>
</table>

<div class="divider-full">&nbsp;</div>

<table border="0"  cellpadding="6" bgcolor="#FFFFFF"> 
<tr>
<td align="center"><img src="brandt.png" class="border" alt="brandt" width="100" height="113" /></td>
<td>
<p class="blue"><b>Linn Brandt</b> (MD) is a hospital physician specializing in internal medicine at Diakonhjemmet Hospital in Oslo, Norway. She is also part-time employed as a researcher at Sykehuset Innlandet Hospital Trust in Gj&#248;vik, Norway, through the MAGIC (Making GRADE the Irresistible Choice) research and innovation program, focusing on the digitizing of clinical practice guidelines and their use in clinical decision support systems. </p>
</td>
</tr>
</table>

<div class="divider-full">&nbsp;</div>

<table border="0"  cellpadding="6" bgcolor="#FFFFFF"> 
<tr>
<td align="center"><img src="mavergames.png" class="border" alt="mavergames" width="100" height="147" /></td>
<td>
<p class="blue"><b>Chris Mavergames</b> (MLIS) is Head of Informatics and Knowledge Management, The Cochrane Collaboration. Chris is charged with leading and improving Cochrane's information, technology, and knowledge management systems. Chris' background is in information science and knowledge management with an MLIS degree from Long Island University in New York, USA. He has more than fourteen years' experience working with XML and related technologies, database management systems, web development technologies, metadata schema and controlled vocabularies and, most recently, linked data and semantic web technologies. Chris began his career at CNN and Time Inc., in the relatively early days of digital and online publishing. He went on to become Director of Multimedia Resources at the New York City College of Technology, and then spent a year working on a digitisation project at the British Library, St. Pancras, London. Chris joined The Cochrane Collaboration in 2006 and took over leadership of the Web Team in 2009. In addition to his work with the Web Team, he is an author in the Cochrane Airways Group and a member of the Information Retrieval Methods Group.
</p>
</td>
</tr>
</table>

<div class="divider-full">&nbsp;</div>

 <!-- Standard Copyright line here  -->

<div class="center">

<p class="footer">Copyright &copy; 2015 Laura Slaughter, Christopher Friis Berntsen, Linn Brandt and Chris Mavergames</p>  
  </div>
</td>
 </tr>
</table>

<table width="100%" border="0" align="center" cellpadding="0" cellspacing="0">
  <tr>
    <td height="1" bgcolor="#2b538e"><img src="../../../img2/transparent.gif" alt="transparent image" width="100" height="2" /></td>
  </tr>
</table>

</td></tr></table>
</td></tr></table>
</form>

</body>
</html>